--- gnome-core-1.4.0.5/help-browser/Makefile.am.orig	Sat Jan 26 19:22:38 2002
+++ gnome-core-1.4.0.5/help-browser/Makefile.am	Sat Jan 26 19:27:41 2002
@@ -93,7 +93,7 @@
 gnorbadir = $(sysconfdir)/CORBA/servers
 gnorba_DATA = help-browser.gnorba
 
-sysdir = $(datadir)/applnk
+sysdir = $(datadir)/applnk/Help
 sys_in_files = gnome-help.desktop.in
 sys_DATA = $(sys_in_files:.desktop.in=.desktop)
 @XML_I18N_MERGE_DESKTOP_RULE@
