--- gnome-core-1.4.0.4/gnome-about/Makefile.am~	Tue Apr 25 02:56:32 2000
+++ gnome-core-1.4.0.4/gnome-about/Makefile.am	Sat Nov 24 23:10:07 2001
@@ -3,6 +3,7 @@
 	-I$(top_srcdir)/intl	\
 	-I$(top_builddir)/intl	\
 	$(GNOME_INCLUDEDIR)	\
+	$(PIXBUF_CFLAGS)	\
 	-DGNOMELOCALEDIR=\""$(datadir)/locale\""
 
 LDADD = \
--- gnome-core-1.4.0.4/applets/gen_util/Makefile.am~	Wed Mar  7 01:09:01 2001
+++ gnome-core-1.4.0.4/applets/gen_util/Makefile.am	Sat Nov 24 23:26:11 2001
@@ -5,7 +5,8 @@
 	-I$(srcdir)/../../panel				\
 	-I$(top_builddir)/panel				\
 	-DGNOMELOCALEDIR=\""$(datadir)/locale"\"	\
-	-I$(includedir) $(GNOME_INCLUDEDIR)
+	-I$(includedir) $(GNOME_INCLUDEDIR)		\
+	$(PIXBUF_CFLAGS)
 
 lib_LTLIBRARIES = libgen_util_applet.la
 
