diff -urN gnome-core-1.4.0.1.org/Makefile.am gnome-core-1.4.0.1/Makefile.am
--- gnome-core-1.4.0.1.org/Makefile.am	Sat Apr  7 14:24:32 2001
+++ gnome-core-1.4.0.1/Makefile.am	Sat Apr  7 15:17:11 2001
@@ -12,7 +12,6 @@
 	 gnome-edit \
 	 gnome-hint \
 	 gnome-about \
-	 core-docs \
 	 panel \
 	 applets \
 	 omf-install \
diff -urN gnome-core-1.4.0.1.org/applets/gen_util/help/es/printer/Makefile.am gnome-core-1.4.0.1/applets/gen_util/help/es/printer/Makefile.am
--- gnome-core-1.4.0.1.org/applets/gen_util/help/es/printer/Makefile.am	Sat Apr  7 14:24:34 2001
+++ gnome-core-1.4.0.1/applets/gen_util/help/es/printer/Makefile.am	Sat Apr  7 14:59:00 2001
@@ -3,6 +3,7 @@
         printer-fig.png\
         printer-properties.png
 lang = es
+omffiles =
 sgml_ents = $(applet).sgml
 include $(top_srcdir)/applet-docs.make
 dist-hook: applet-dist-hook
diff -urN gnome-core-1.4.0.1.org/gnome-terminal/da/Makefile.am gnome-core-1.4.0.1/gnome-terminal/da/Makefile.am
--- gnome-core-1.4.0.1.org/gnome-terminal/da/Makefile.am	Sat Apr  7 14:24:31 2001
+++ gnome-core-1.4.0.1/gnome-terminal/da/Makefile.am	Sat Apr  7 14:59:00 2001
@@ -1,6 +1,7 @@
 figs =
 app = gnome-terminal
 lang = da
+omffiles =
 # Commenting out for now b/c TOC extraction is broken for this SGML file
 # omffiles = gnome-terminal-da.omf
 include $(top_srcdir)/app-docs.make
diff -urN gnome-core-1.4.0.1.org/gnome-terminal/it/Makefile.am gnome-core-1.4.0.1/gnome-terminal/it/Makefile.am
--- gnome-core-1.4.0.1.org/gnome-terminal/it/Makefile.am	Sat Apr  7 14:24:31 2001
+++ gnome-core-1.4.0.1/gnome-terminal/it/Makefile.am	Sat Apr  7 14:59:00 2001
@@ -1,6 +1,7 @@
 figs =
 app = gnome-terminal
 lang = it
+omffiles =
 # Commenting out for now b/c TOC extraction is broken for this SGML file
 #omffiles = gnome-terminal-it.omf
 include $(top_srcdir)/app-docs.make
diff -urN gnome-core-1.4.0.1.org/gnome-terminal/no/Makefile.am gnome-core-1.4.0.1/gnome-terminal/no/Makefile.am
--- gnome-core-1.4.0.1.org/gnome-terminal/no/Makefile.am	Sat Apr  7 14:24:31 2001
+++ gnome-core-1.4.0.1/gnome-terminal/no/Makefile.am	Sat Apr  7 14:59:00 2001
@@ -1,6 +1,7 @@
 figs =
 app = gnome-terminal
 lang = no
+omffiles =
 # Commenting out for now b/c TOC extraction is broken for this SGML file
 # omffiles = gnome-terminal-no.omf
 include $(top_srcdir)/app-docs.make
diff -urN gnome-core-1.4.0.1.org/sgmldocs.make gnome-core-1.4.0.1/sgmldocs.make
--- gnome-core-1.4.0.1.org/sgmldocs.make	Sat Apr  7 14:24:30 2001
+++ gnome-core-1.4.0.1/sgmldocs.make	Sat Apr  7 14:59:00 2001
@@ -58,7 +58,7 @@
 	-cp $(docname)/index.html .
 
 $(docname).sgml: $(sgml_ents)
-        -ourdir=`pwd`;  \
+        ourdir=`pwd`;  \
         cd $(srcdir);   \
         cp $(sgml_ents) $$ourdir
 
diff -urN gnome-core-1.4.0.1.org/smproxy/Makefile.am gnome-core-1.4.0.1/smproxy/Makefile.am
--- gnome-core-1.4.0.1.org/smproxy/Makefile.am	Sat Apr  7 14:24:30 2001
+++ gnome-core-1.4.0.1/smproxy/Makefile.am	Sat Apr  7 14:59:14 2001
@@ -13,6 +13,4 @@
 
 gnome_smproxy_SOURCES = save.c smproxy.h smproxy.c
 
-man1_MANS = smproxy.man
-
 EXTRA_DIST = README.gnome
