diff -Nru gnome-core-1.2.0/desktop-links/Makefile.am gnome-core-1.2.0.new/desktop-links/Makefile.am
--- gnome-core-1.2.0/desktop-links/Makefile.am	Sun May 28 03:07:09 2000
+++ gnome-core-1.2.0.new/desktop-links/Makefile.am	Sun May 28 03:03:13 2000
@@ -1,4 +1,4 @@
-appsdir = $(datadir)/gnome/apps
+appsdir = $(datadir)/applnk
 
 Rootdir           = $(appsdir)
 Applicationsdir   = $(appsdir)/Applications
diff -Nru gnome-core-1.2.0/gmenu/Makefile.am gnome-core-1.2.0.new/gmenu/Makefile.am
--- gnome-core-1.2.0/gmenu/Makefile.am	Sun May 28 03:07:09 2000
+++ gnome-core-1.2.0.new/gmenu/Makefile.am	Sun May 28 03:03:13 2000
@@ -26,5 +26,5 @@
 
 EXTRA_DIST = gmenu.desktop top.xpm unknown.xpm
 
-sysdir = $(datadir)/gnome/apps/Settings
+sysdir = $(datadir)/applnk/Settings
 sys_DATA = gmenu.desktop
diff -Nru gnome-core-1.2.0/gmenu/main.c gnome-core-1.2.0.new/gmenu/main.c
--- gnome-core-1.2.0/gmenu/main.c	Sun May 28 03:07:09 2000
+++ gnome-core-1.2.0.new/gmenu/main.c	Sun May 28 03:03:13 2000
@@ -276,7 +276,7 @@
 	gnome_init ("GNOME menu editor", VERSION, argc, argv);
 	gnome_window_icon_set_default_from_file (GNOME_ICONDIR"/gnome-gmenu.png");
 
-	system_apps_dir = gnome_datadir_file("gnome/apps");
+	system_apps_dir = gnome_datadir_file("applnk");
 	system_applets_dir = gnome_datadir_file("applets");
 	system_pixmap_dir = gnome_datadir_file("pixmaps");
 	if (!system_apps_dir || !system_pixmap_dir || !system_applets_dir)
diff -Nru gnome-core-1.2.0/gnome-hint/Makefile.am gnome-core-1.2.0.new/gnome-hint/Makefile.am
--- gnome-core-1.2.0/gnome-hint/Makefile.am	Sun May 28 03:07:09 2000
+++ gnome-core-1.2.0.new/gnome-hint/Makefile.am	Sun May 28 03:07:42 2000
@@ -23,7 +23,7 @@
 	$(GNOME_CAPPLET_LIBS)	\
 	$(INTLLIBS)
 
-utildir = $(datadir)/gnome/apps/Utilities
+utildir = $(datadir)/applnk/Utilities
 util_DATA = gnome-hint.desktop
 
 EXTRA_DIST = \
@@ -37,7 +37,5 @@
 pixmapsdir = $(datadir)/pixmaps
 pixmaps_DATA = gnome-hint.png
 
-install-data-local:
-	$(mkinstalldirs) $(DESTDIR)$(datadir)/gnome/apps/Settings/Session/
-	$(INSTALL_DATA) $(srcdir)/gnome-hint-properties.desktop $(DESTDIR)$(datadir)/gnome/apps/Settings/Session/gnome-hint-properties.desktop
-
+desktopdir = $(datadir)/applnk/Settings/Session
+desktop_DATA = gnome-hint-properties.desktop
diff -Nru gnome-core-1.2.0/gnome-terminal/Makefile.am gnome-core-1.2.0.new/gnome-terminal/Makefile.am
--- gnome-core-1.2.0/gnome-terminal/Makefile.am	Sun May 28 03:07:09 2000
+++ gnome-core-1.2.0.new/gnome-terminal/Makefile.am	Sun May 28 03:03:13 2000
@@ -48,5 +48,5 @@
 gnorbadir = $(sysconfdir)/CORBA/servers
 gnorba_DATA = gnome-terminal.gnorba
 
-utildir = $(datadir)/gnome/apps/System
+utildir = $(datadir)/applnk/System
 util_DATA = gnome-terminal.desktop
diff -Nru gnome-core-1.2.0/gsm/Makefile.am gnome-core-1.2.0.new/gsm/Makefile.am
--- gnome-core-1.2.0/gsm/Makefile.am	Sun May 28 03:07:09 2000
+++ gnome-core-1.2.0.new/gsm/Makefile.am	Sun May 28 03:03:13 2000
@@ -11,7 +11,7 @@
 
 LDADD = $(GNOME_LIBDIR) $(GNOMEUI_LIBS) $(INTLLIBS)
 
-sysdir = $(datadir)/apps/Settings
+sysdir = $(datadir)/applnk/Settings
 
 if SESSION
 noinst_DATA = session.desktop
@@ -36,7 +36,7 @@
 
 default_DATA = default.session default.wm
 
-desktopdir = $(datadir)/gnome/apps/Settings/Session/
+desktopdir = $(datadir)/applnk/Settings/Session/
 desktop_DATA = save-session.desktop session.desktop
 
 default.session: default.in ../config.status
diff -Nru gnome-core-1.2.0/help-browser/Makefile.am gnome-core-1.2.0.new/help-browser/Makefile.am
--- gnome-core-1.2.0/help-browser/Makefile.am	Sun May 28 03:07:09 2000
+++ gnome-core-1.2.0.new/help-browser/Makefile.am	Sun May 28 03:03:13 2000
@@ -89,7 +89,7 @@
 gnorbadir = $(sysconfdir)/CORBA/servers
 gnorba_DATA = help-browser.gnorba
 
-sysdir = $(datadir)/gnome/apps
+sysdir = $(datadir)/applnk
 sys_DATA = gnome-help.desktop
 
 install-exec-local:
diff -Nru gnome-core-1.2.0/panel/Makefile.am gnome-core-1.2.0.new/panel/Makefile.am
--- gnome-core-1.2.0/panel/Makefile.am	Sun May 28 03:07:09 2000
+++ gnome-core-1.2.0.new/panel/Makefile.am	Sun May 28 03:03:13 2000
@@ -87,8 +87,6 @@
 	menu.h  		\
 	menu-fentry.c  		\
 	menu-fentry.h  		\
-	menu-rh.c  		\
-	menu-rh.h  		\
 	menu-properties.c	\
 	menu-properties.h	\
 	menu-util.c		\
@@ -227,7 +225,7 @@
 sysdir = $(datadir)/control-center/Desktop/
 sys_DATA = gnome-panel-properties.desktop
 
-deskdir = $(datadir)/gnome/apps/Settings/Desktop/
+deskdir = $(datadir)/applnk/Settings/Desktop/
 desk_DATA = gnome-panel-properties.desktop
 
 $(srcdir)/panel-util.c: $(CORBA_SRCLIST)
diff -Nru gnome-core-1.2.0/panel/gnome-panel-properties.c gnome-core-1.2.0.new/panel/gnome-panel-properties.c
--- gnome-core-1.2.0/panel/gnome-panel-properties.c	Sun May 28 03:07:09 2000
+++ gnome-core-1.2.0.new/panel/gnome-panel-properties.c	Sun May 28 03:03:13 2000
@@ -109,9 +109,6 @@
 	{ MAIN_MENU_SYSTEM,  MAIN_MENU_SYSTEM_SUB,  N_("Programs: ") },
 	{ MAIN_MENU_USER,    MAIN_MENU_USER_SUB,    N_("Favorites: ") },
 	{ MAIN_MENU_APPLETS, MAIN_MENU_APPLETS_SUB, N_("Applets: ") },
-	{ MAIN_MENU_REDHAT,  MAIN_MENU_REDHAT_SUB,  N_("AnotherLevel: ") },
-	{ MAIN_MENU_KDE,     MAIN_MENU_KDE_SUB,     N_("KDE: ") },
-	{ MAIN_MENU_DEBIAN,  MAIN_MENU_DEBIAN_SUB,  N_("Debian: ") },
 	{ MAIN_MENU_PANEL,   MAIN_MENU_PANEL_SUB,   N_("Panel: ") },
 	{ MAIN_MENU_DESKTOP, MAIN_MENU_DESKTOP_SUB, N_("Desktop: ") },
 	{ 0 }
@@ -1108,8 +1105,7 @@
 
 	g_string_sprintf(buf,"menu_flags=%d", 
 			 (int)(MAIN_MENU_SYSTEM_SUB | MAIN_MENU_USER_SUB|
-			       MAIN_MENU_APPLETS_SUB | MAIN_MENU_REDHAT_SUB|
-			       MAIN_MENU_DEBIAN_SUB | MAIN_MENU_KDE_SUB|
+			       MAIN_MENU_APPLETS_SUB |
 			       MAIN_MENU_PANEL | MAIN_MENU_DESKTOP));
 	global_config.menu_flags = gnome_config_get_int(buf->str);
 
diff -Nru gnome-core-1.2.0/panel/main.c gnome-core-1.2.0.new/panel/main.c
--- gnome-core-1.2.0/panel/main.c	Sun May 28 03:07:09 2000
+++ gnome-core-1.2.0.new/panel/main.c	Sun May 28 03:03:13 2000
@@ -44,10 +44,6 @@
 /*a list of started extern applet child processes*/
 extern GList * children;
 
-char *kde_menudir = NULL;
-char *kde_icondir = NULL;
-char *kde_mini_icondir = NULL;
-
 static int
 menu_age_timeout(gpointer data)
 {
@@ -98,52 +94,6 @@
 }
 
 static void
-find_kde_directory(void)
-{
-	int i;
-	char *kdedir = getenv("KDEDIR");
-	char *try_prefixes[] = {
-		"/usr",
-		"/opt/kde",
-		"/usr/local",
-		"/kde",
-		NULL
-	};
-	if(kdedir) {
-		kde_menudir = g_concat_dir_and_file(kdedir,"share/applnk");
-		kde_icondir = g_concat_dir_and_file(kdedir,"share/icons");
-		kde_mini_icondir = g_concat_dir_and_file(kdedir,"share/icons/mini");
-		return;
-	}
-
-	/* if what configure gave us works use that */
-	if(g_file_test(KDE_MENUDIR,G_FILE_TEST_ISDIR)) {
-		kde_menudir = g_strdup(KDE_MENUDIR);
-		kde_icondir = g_strdup(KDE_ICONDIR);
-		kde_mini_icondir = g_strdup(KDE_MINI_ICONDIR);
-		return;
-	}
-
-	for(i=0;try_prefixes[i];i++) {
-		char *try;
-		try = g_concat_dir_and_file(try_prefixes[i],"share/applnk");
-		if(g_file_test(try,G_FILE_TEST_ISDIR)) {
-			kde_menudir = try;
-			kde_icondir = g_concat_dir_and_file(try_prefixes[i],"share/icons");
-			kde_mini_icondir = g_concat_dir_and_file(try_prefixes[i],"share/icons/mini");
-			return;
-		}
-		g_free(try);
-	}
-
-	/* absolute fallback, these don't exist, but we're out of options
-	   here */
-	kde_menudir = g_strdup(KDE_MENUDIR);
-	kde_icondir = g_strdup(KDE_ICONDIR);
-	kde_mini_icondir = g_strdup(KDE_MINI_ICONDIR);
-}
-
-static void
 setup_visuals (void)
 {
 	gdk_rgb_init ();
@@ -195,8 +145,6 @@
 		break;
 	}
 	}
-
-	find_kde_directory();
 
 	client = gnome_master_client ();
 
diff -Nru gnome-core-1.2.0/panel/menu-properties.c gnome-core-1.2.0.new/panel/menu-properties.c
--- gnome-core-1.2.0/panel/menu-properties.c	Sun May 28 03:07:09 2000
+++ gnome-core-1.2.0.new/panel/menu-properties.c	Sun May 28 03:03:13 2000
@@ -40,7 +40,7 @@
 		arguments = ".";
 
 	if(strcmp(arguments,".")==0)
-		this_menu = gnome_unconditional_datadir_file ("gnome/apps");
+		this_menu = gnome_unconditional_datadir_file ("applnk");
 	else if (*arguments == '/')
 		this_menu = g_strdup (arguments);
 	else if (*arguments == '~')
@@ -97,12 +97,6 @@
 	GtkWidget *user_sub = gtk_object_get_data(GTK_OBJECT(widget), "user_sub");
 	GtkWidget *applets = gtk_object_get_data(GTK_OBJECT(widget), "applets");
 	GtkWidget *applets_sub = gtk_object_get_data(GTK_OBJECT(widget), "applets_sub");
-	GtkWidget *redhat = gtk_object_get_data(GTK_OBJECT(widget), "redhat");
-	GtkWidget *redhat_sub = gtk_object_get_data(GTK_OBJECT(widget), "redhat_sub");
- 	GtkWidget *kde = gtk_object_get_data(GTK_OBJECT(widget), "kde");
- 	GtkWidget *kde_sub = gtk_object_get_data(GTK_OBJECT(widget), "kde_sub");
-	GtkWidget *debian = gtk_object_get_data(GTK_OBJECT(widget), "debian");
-	GtkWidget *debian_sub = gtk_object_get_data(GTK_OBJECT(widget), "debian_sub");
 
 	GtkWidget *panel = gtk_object_get_data(GTK_OBJECT(widget), "panel");
 	GtkWidget *panel_sub = gtk_object_get_data(GTK_OBJECT(widget), "panel_sub");
@@ -155,21 +149,6 @@
 	else if (GTK_TOGGLE_BUTTON (applets)->active)
 		menu->main_menu_flags |= MAIN_MENU_APPLETS;
 
-	if(GTK_TOGGLE_BUTTON(redhat_sub)->active)
-		menu->main_menu_flags |= MAIN_MENU_REDHAT_SUB;
-	else if (GTK_TOGGLE_BUTTON (redhat)->active)
-		menu->main_menu_flags |= MAIN_MENU_REDHAT;
-
-	if(GTK_TOGGLE_BUTTON(debian_sub)->active)
-		menu->main_menu_flags |= MAIN_MENU_DEBIAN_SUB;
-	else if (GTK_TOGGLE_BUTTON (debian)->active)
-		menu->main_menu_flags |= MAIN_MENU_DEBIAN;
-
-	if(GTK_TOGGLE_BUTTON(kde_sub)->active)
-		menu->main_menu_flags |= MAIN_MENU_KDE_SUB;
-	else if (GTK_TOGGLE_BUTTON (kde)->active)
-		menu->main_menu_flags |= MAIN_MENU_KDE;
-
 	if(GTK_TOGGLE_BUTTON(panel_sub)->active)
 		menu->main_menu_flags |= MAIN_MENU_PANEL_SUB;
 	else if (GTK_TOGGLE_BUTTON(panel)->active)
@@ -371,22 +350,10 @@
 			      menu->main_menu_flags&MAIN_MENU_APPLETS,
 			      menu->main_menu_flags&MAIN_MENU_APPLETS_SUB);
 	add_menu_type_options(GTK_OBJECT(dialog),GTK_TABLE(table),3,
-			      _("AnotherLevel menu (if found): "),"redhat",
-			      menu->main_menu_flags&MAIN_MENU_REDHAT,
-			      menu->main_menu_flags&MAIN_MENU_REDHAT_SUB);
- 	add_menu_type_options(GTK_OBJECT(dialog),GTK_TABLE(table),4,
- 			      _("KDE menu (if found): "),"kde",
- 			      menu->main_menu_flags&MAIN_MENU_KDE,
- 			      menu->main_menu_flags&MAIN_MENU_KDE_SUB);
-	add_menu_type_options(GTK_OBJECT(dialog),GTK_TABLE(table),5,
-			      _("Debian menu (if found): "),"debian",
-			      menu->main_menu_flags&MAIN_MENU_DEBIAN,
-			      menu->main_menu_flags&MAIN_MENU_DEBIAN_SUB);
-	add_menu_type_options(GTK_OBJECT(dialog),GTK_TABLE(table),6,
 			      _("Panel menu: "),"panel",
 			      menu->main_menu_flags&MAIN_MENU_PANEL,
 			      menu->main_menu_flags&MAIN_MENU_PANEL_SUB);
-	add_menu_type_options(GTK_OBJECT(dialog),GTK_TABLE(table),7,
+	add_menu_type_options(GTK_OBJECT(dialog),GTK_TABLE(table),4,
 			      _("Desktop menu: "),"desktop",
 			      menu->main_menu_flags&MAIN_MENU_DESKTOP,
 			      menu->main_menu_flags&MAIN_MENU_DESKTOP_SUB);
diff -Nru gnome-core-1.2.0/panel/menu-util.c gnome-core-1.2.0.new/panel/menu-util.c
--- gnome-core-1.2.0/panel/menu-util.c	Sun May 28 03:07:09 2000
+++ gnome-core-1.2.0.new/panel/menu-util.c	Sun May 28 03:03:13 2000
@@ -13,10 +13,6 @@
 
 /*#define PANEL_DEBUG 1*/
 
-extern char *kde_menudir;
-extern char *kde_icondir;
-extern char *kde_mini_icondir;
-
 void
 add_menu_separator (GtkWidget *menu)
 {
@@ -96,18 +92,6 @@
 	int flags = MAIN_MENU_SYSTEM_SUB | MAIN_MENU_USER_SUB |
 		MAIN_MENU_APPLETS_SUB | MAIN_MENU_PANEL_SUB |
 		MAIN_MENU_DESKTOP;
-	
-	/*guess redhat menus*/
-	if(g_file_exists(REDHAT_MENUDIR))
-		flags |= MAIN_MENU_REDHAT_SUB;
-
-	/*guess KDE menus*/
-	if(g_file_exists(kde_menudir))
-		flags |= MAIN_MENU_KDE_SUB;
-
-	/*guess debian menus*/
-	if (g_file_exists(DEBIAN_MENUDIR))
-		flags |= MAIN_MENU_DEBIAN_SUB;
 	
 	return flags;
 }
diff -Nru gnome-core-1.2.0/panel/menu.c gnome-core-1.2.0.new/panel/menu.c
--- gnome-core-1.2.0/panel/menu.c	Sun May 28 03:07:09 2000
+++ gnome-core-1.2.0.new/panel/menu.c	Sun May 28 03:03:13 2000
@@ -55,10 +55,6 @@
 
 extern int base_panels;
 
-extern char *kde_menudir;
-extern char *kde_icondir;
-extern char *kde_mini_icondir;
-
 extern GtkTooltips *panel_tooltips;
 
 typedef struct _TearoffMenu TearoffMenu;
@@ -82,7 +78,7 @@
 	/*just load the menus from disk, don't make the widgets
 	  this just reads the .desktops of the top most directory
 	  and a level down*/
-	char *menu = gnome_datadir_file("gnome/apps");
+	char *menu = gnome_datadir_file("applnk");
 	if(menu)
 		fr_read_dir(NULL,menu,NULL,2);
 	g_free(menu);
@@ -90,13 +86,10 @@
 	if(menu)
 		fr_read_dir(NULL,menu,NULL,2);
 	g_free(menu);
-	menu = gnome_util_home_file("apps");
+	menu = gnome_util_home_file("applnk");
 	if(menu)
 		fr_read_dir(NULL,menu,NULL,2);
 	g_free(menu);
-	/*if redhat menus, use the fork version to read*/
-	if(g_file_exists(REDHAT_MENUDIR))
-		create_rh_menu(TRUE);
 }
 
 /*the most important dialog in the whole application*/
@@ -185,7 +178,7 @@
 	char *argv[6]={"cp","-r","-f"};
 	char *p;
 
-	p = gnome_util_home_file("apps");
+	p = gnome_util_home_file("applnk");
 	argv[3] = item_loc;
 	argv[4] = p;
 	argv[5] = NULL;
@@ -866,18 +859,6 @@
 		MAIN_MENU_APPLETS_SUB | MAIN_MENU_PANEL_SUB |
 		MAIN_MENU_DESKTOP_SUB;
 	
-	/*guess redhat menus*/
-	if (g_file_exists(REDHAT_MENUDIR))
-		flags |= MAIN_MENU_REDHAT_SUB;
-
-	/*guess KDE menus*/
-	if(g_file_exists(kde_menudir))
-		flags |= MAIN_MENU_KDE_SUB;
-
-	/*guess debian menus*/
-	if (g_file_exists(DEBIAN_MENUDIR))
-		flags |= MAIN_MENU_DEBIAN_SUB;
-
 	panel = get_panel_from_menu_data (widget);
 
 	load_menu_applet (menudir, flags, panel, 0, FALSE);
@@ -1130,7 +1111,7 @@
 						   GTK_SIGNAL_FUNC(add_app_to_personal),
 						   sim->item_loc);
 				/*ummmm slightly ugly but should work 99% of time*/
-				if(strstr(sim->item_loc,"/.gnome/apps/"))
+				if(strstr(sim->item_loc,"/.applnk/"))
 					gtk_widget_set_sensitive(menuitem,FALSE);
 			}
 
@@ -1177,7 +1158,7 @@
 					   GTK_SIGNAL_FUNC(add_app_to_personal),
 					   sim->mf->menudir);
 			/*ummmm slightly ugly but should work 99% of time*/
-			if(strstr(sim->mf->menudir, "/.gnome/apps"))
+			if(strstr(sim->mf->menudir, "/.applnk"))
 				gtk_widget_set_sensitive(menuitem, FALSE);
 
 			menuitem = gtk_menu_item_new ();
@@ -1222,9 +1203,6 @@
 	
 	gtk_widget_set_sensitive(sim->prop_item,FALSE);
 	if(sim->item_loc &&
-	   /*A HACK: but it works, don't have it edittable if it's redhat
-	     menus as they are auto generated!*/
-	   !strstr(sim->item_loc,"/.gnome/apps-redhat/") &&
 	   /*if it's a kdelnk file, don't let it be editted*/
 	   !is_ext(sim->item_loc,".kdelnk") &&
 	   access(sim->item_loc,W_OK)==0) {
@@ -1236,10 +1214,7 @@
 	} else if(!sim->item_loc || errno==ENOENT) {
 		/*the dentry isn't there, check if we can write the
 		  directory*/
-		if(access(sim->mf->menudir,W_OK)==0 &&
-		   /*A HACK: but it works, don't have it edittable if it's redhat
-		     menus as they are auto generated!*/
-		   !strstr(sim->mf->menudir,".gnome/apps-redhat/"))
+		if(access(sim->mf->menudir,W_OK)==0)
 			gtk_widget_set_sensitive(sim->prop_item,TRUE);
 	}
 	
@@ -2709,7 +2684,7 @@
 create_system_menu(GtkWidget *menu, gboolean fake_submenus, gboolean fake,
 		   gboolean title)
 {
-	char *menudir = gnome_datadir_file ("gnome/apps");
+	char *menudir = gnome_datadir_file ("applnk");
 
 	if (menudir &&
 	    g_file_test (menudir, G_FILE_TEST_ISDIR)) {
@@ -2764,47 +2739,6 @@
 	return menu;
 }
 
-static GtkWidget *
-create_debian_menu(GtkWidget *menu, gboolean fake_submenus, gboolean fake,
-		   gboolean title)
-{
-	if (!fake || menu) {
-		menu = create_menu_at (menu, DEBIAN_MENUDIR, FALSE,
-				       _("Debian menus"), "gnome-debian.png",
-				       fake_submenus, FALSE, title);
-	} else {
-		menu = create_fake_menu_at (DEBIAN_MENUDIR, FALSE,
-					    _("Debian menus"),
-					    "gnome-debian.png", title);
-	}
-
-	return menu;
-}
-
-static GtkWidget *
-create_kde_menu(GtkWidget *menu, gboolean fake_submenus,
-		gboolean force, gboolean fake, gboolean title)
-{
-	char *pixmap_name;
-
-	pixmap_name = g_concat_dir_and_file (kde_icondir, "exec.xpm");
-
-	if(!fake || menu) {
-		menu = create_menu_at (menu, 
-				       kde_menudir, FALSE,
-				       _("KDE menus"), 
-				       pixmap_name,
-				       fake_submenus,
-				       force, title);
-	} else {
-		menu = create_fake_menu_at (kde_menudir, FALSE,
-					    _("KDE menus"),
-					    pixmap_name, title);
-	}
-	g_free (pixmap_name);
-	return menu;
-}
-
 static void
 status_unparent(GtkWidget *widget)
 {
@@ -3598,8 +3532,8 @@
 	gtk_menu_append (GTK_MENU (submenu), submenuitem);
 	gtk_signal_connect(GTK_OBJECT(submenuitem), "activate",
 			   GTK_SIGNAL_FUNC(add_menu_to_panel),
-			   "gnome/apps/");
-	setup_internal_applet_drag(submenuitem, "MENU:gnome/apps/");
+			   "applnk/");
+	setup_internal_applet_drag(submenuitem, "MENU:applnk/");
 
 	submenuitem = gtk_menu_item_new ();
 	gtk_widget_lock_accelerators (menuitem);
@@ -3608,8 +3542,8 @@
 	gtk_menu_append (GTK_MENU (submenu), submenuitem);
 	gtk_signal_connect(GTK_OBJECT(submenuitem), "activate",
 			   GTK_SIGNAL_FUNC(add_menu_to_panel),
-			   "~/.gnome/apps/");
-	setup_internal_applet_drag(submenuitem, "MENU:~/.gnome/apps/");
+			   "~/.applnk/");
+	setup_internal_applet_drag(submenuitem, "MENU:~/.applnk/");
 
 	menuitem = gtk_menu_item_new ();
 	gtk_widget_lock_accelerators (menuitem);
@@ -4073,17 +4007,11 @@
 
 	gboolean has_inline = (flags & (MAIN_MENU_SYSTEM |
 					MAIN_MENU_USER |
-					MAIN_MENU_APPLETS |
-					MAIN_MENU_KDE |
-					MAIN_MENU_REDHAT |
-					MAIN_MENU_DEBIAN));
+					MAIN_MENU_APPLETS));
 
 	gboolean has_subs = (flags & (MAIN_MENU_SYSTEM_SUB |
 				      MAIN_MENU_USER_SUB |
-				      MAIN_MENU_APPLETS_SUB |
-				      MAIN_MENU_KDE_SUB |
-				      MAIN_MENU_REDHAT_SUB |
-				      MAIN_MENU_DEBIAN_SUB));
+				      MAIN_MENU_APPLETS_SUB));
 
 	gboolean has_inline2 = (flags & (MAIN_MENU_DESKTOP |
 					 MAIN_MENU_PANEL));
@@ -4107,24 +4035,12 @@
 	if (flags & MAIN_MENU_SYSTEM)
 		create_system_menu(root_menu, fake_submenus, FALSE, title);
 	if (flags & MAIN_MENU_USER)
-		create_user_menu(_("Favorites"), "apps",
+		create_user_menu(_("Favorites"), "applnk",
 				 root_menu, "gnome-favorites.png",
 				 fake_submenus, FALSE, FALSE, title);
 	if (flags & MAIN_MENU_APPLETS)
 		create_applets_menu(root_menu, fake_submenus, title);
 
-	if (flags & MAIN_MENU_REDHAT) {
-		rh_submenu_to_display(NULL,NULL);
-		create_user_menu(_("AnotherLevel menus"), "apps-redhat",
-				 root_menu, NULL, fake_submenus, FALSE,
-				 FALSE, title);
-	}
-
-	if (flags & MAIN_MENU_DEBIAN)
-		create_debian_menu(root_menu, fake_submenus, FALSE, title);
-	if (flags & MAIN_MENU_KDE)
-		create_kde_menu(root_menu, fake_submenus, FALSE, FALSE, title);
-
 	/*others here*/
 
 	if (has_subs && has_inline)
@@ -4149,7 +4065,7 @@
 	}
 
 	if (flags & MAIN_MENU_USER_SUB) {
-		menu = create_user_menu(_("Favorites"), "apps", NULL,
+		menu = create_user_menu(_("Favorites"), "applnk", NULL,
 					"gnome-favorites.png",
 					fake_submenus, TRUE, TRUE, TRUE);
 		menuitem = gtk_menu_item_new ();
@@ -4174,61 +4090,6 @@
 				   GTK_SIGNAL_FUNC(submenu_to_display),
 				   NULL);
 	}
-	if (flags & MAIN_MENU_REDHAT_SUB) {
-		menu = create_user_menu(_("AnotherLevel menus"), "apps-redhat",
-					NULL, NULL, fake_submenus, TRUE, TRUE,
-					TRUE);
-		menuitem = gtk_menu_item_new ();
-		gtk_widget_lock_accelerators (menuitem);
-		setup_menuitem_with_size (menuitem, NULL, 
-					  _("AnotherLevel menus"),
-					  size);
-		gtk_menu_append (GTK_MENU (root_menu), menuitem);
-		gtk_menu_item_set_submenu (GTK_MENU_ITEM (menuitem), menu);
-		gtk_signal_connect(GTK_OBJECT(menu),"show",
-				   GTK_SIGNAL_FUNC(rh_submenu_to_display),
-				   menuitem);
-		gtk_signal_connect(GTK_OBJECT(menu),"show",
-				   GTK_SIGNAL_FUNC(submenu_to_display), NULL);
-	}
-	if (flags & MAIN_MENU_DEBIAN_SUB) {
-		menu = create_debian_menu(NULL, fake_submenus, TRUE, TRUE);
-		menuitem = gtk_menu_item_new ();
-		gtk_widget_lock_accelerators (menuitem);
-		setup_menuitem_try_pixmap (menuitem,
-					   "gnome-debian.png",
-					   _("Debian menus"), size);
-		gtk_menu_append (GTK_MENU (root_menu), menuitem);
-		if(menu) {
-			gtk_menu_item_set_submenu (GTK_MENU_ITEM (menuitem),
-						   menu);
-			gtk_signal_connect(GTK_OBJECT(menu),"show",
-					   GTK_SIGNAL_FUNC(submenu_to_display),
-					   NULL);
-		}
-	}
-	if (flags & MAIN_MENU_KDE_SUB) {
-		GtkWidget *pixmap = NULL;
-		char *pixmap_path;
-		menu = create_kde_menu(NULL, fake_submenus, TRUE, TRUE, TRUE);
-		pixmap_path = g_concat_dir_and_file (kde_icondir, "exec.xpm");
-
-		if (g_file_exists(pixmap_path)) {
-			pixmap = fake_pixmap_at_size (pixmap_path, size);
-			if (pixmap)
-				gtk_widget_show (pixmap);
-		}
-		g_free (pixmap_path);
-		menuitem = gtk_menu_item_new ();
-		gtk_widget_lock_accelerators (menuitem);
-		setup_menuitem_with_size (menuitem, pixmap, _("KDE menus"),
-					  size);
-		gtk_menu_append (GTK_MENU (root_menu), menuitem);
-		gtk_menu_item_set_submenu (GTK_MENU_ITEM (menuitem), menu);
-		gtk_signal_connect(GTK_OBJECT(menu),"show",
-				   GTK_SIGNAL_FUNC(submenu_to_display), NULL);
-	}
-
 	menuitem = gtk_menu_item_new ();
 	gtk_widget_lock_accelerators (menuitem);
 	setup_menuitem_try_pixmap (menuitem, "gnome-run.png", 
@@ -4347,10 +4208,6 @@
 
 		g_slist_free(list);
 	} else {
-		if(menu->main_menu_flags&MAIN_MENU_REDHAT &&
-		   !(menu->main_menu_flags&MAIN_MENU_REDHAT_SUB))
-			rh_submenu_to_display(NULL,NULL);
-
 		check_and_reread_applet(menu, main_menu);
 	}
 
diff -Nru gnome-core-1.2.0/panel/menu.h gnome-core-1.2.0.new/panel/menu.h
--- gnome-core-1.2.0/panel/menu.h	Sun May 28 03:07:09 2000
+++ gnome-core-1.2.0.new/panel/menu.h	Sun May 28 03:03:13 2000
@@ -18,18 +18,12 @@
 	MAIN_MENU_USER_SUB = 1<<1,
 	MAIN_MENU_SYSTEM = 1<<2,
 	MAIN_MENU_SYSTEM_SUB = 1<<3,
-	MAIN_MENU_REDHAT = 1<<4,
-	MAIN_MENU_REDHAT_SUB = 1<<5,
-	MAIN_MENU_KDE = 1<<6,
-	MAIN_MENU_KDE_SUB = 1<<7,
-	MAIN_MENU_DEBIAN = 1<<8,
-	MAIN_MENU_DEBIAN_SUB = 1<<9,
-	MAIN_MENU_APPLETS = 1<<10,
-	MAIN_MENU_APPLETS_SUB = 1<<11,
-	MAIN_MENU_PANEL = 1<<12,
-	MAIN_MENU_PANEL_SUB = 1<<13,
-	MAIN_MENU_DESKTOP = 1<<14,
-	MAIN_MENU_DESKTOP_SUB = 1<<15
+	MAIN_MENU_APPLETS = 1<<4,
+	MAIN_MENU_APPLETS_SUB = 1<<5,
+	MAIN_MENU_PANEL = 1<<6,
+	MAIN_MENU_PANEL_SUB = 1<<7,
+	MAIN_MENU_DESKTOP = 1<<8,
+	MAIN_MENU_DESKTOP_SUB = 1<<9
 };
 
 typedef struct _Menu Menu;
@@ -92,8 +86,6 @@
 #define MENU_PATH "menu_path"
 
 #define MENU_PROPERTIES "menu_properties"
-
-#define DEBIAN_MENUDIR "/var/lib/gnome/Debian/."
 
 #define MENU_TYPES "types_menu"
 #define MENU_TYPE_EDGE "type-edge"
diff -Nru gnome-core-1.2.0/panel/panel.c gnome-core-1.2.0.new/panel/panel.c
--- gnome-core-1.2.0/panel/panel.c	Sun May 28 03:07:09 2000
+++ gnome-core-1.2.0.new/panel/panel.c	Sun May 28 03:03:13 2000
@@ -45,11 +45,6 @@
 
 extern GtkTooltips *panel_tooltips;
 
-extern char *kde_menudir;
-extern char *kde_icondir;
-extern char *kde_mini_icondir;
-
-
 /*the types of stuff we accept*/
 
 enum {
@@ -854,12 +849,6 @@
 {
 	int flags = MAIN_MENU_SYSTEM|MAIN_MENU_USER;
 
-	/*guess redhat menus*/
-	if(g_file_exists("/etc/X11/wmconfig"))
-		flags |= MAIN_MENU_REDHAT_SUB;
-	/* Guess KDE menus */
-	if(g_file_exists(kde_menudir))
-		flags |= MAIN_MENU_KDE_SUB;
 	load_menu_applet(dir, flags, panel, pos, TRUE);
 }
 
diff -Nru gnome-core-1.2.0/panel/session.c gnome-core-1.2.0.new/panel/session.c
--- gnome-core-1.2.0/panel/session.c	Sun May 28 03:07:09 2000
+++ gnome-core-1.2.0.new/panel/session.c	Sun May 28 03:03:13 2000
@@ -42,10 +42,6 @@
 /*list of all panel widgets created*/
 extern GSList *panel_list;
 
-extern char *kde_menudir;
-extern char *kde_icondir;
-extern char *kde_mini_icondir;
-
 int ss_cur_applet = 0;
 int ss_done_save = FALSE;
 gushort ss_cookie = 0;
@@ -777,10 +773,10 @@
 load_default_applets1(PanelWidget *panel)
 {
 	char *def_launchers[] =
-	      { "gnome/apps/gnome-help.desktop",
-		"gnome/apps/System/gnome-terminal.desktop",
-		"gnome/apps/Settings/gnomecc.desktop",
-		"gnome/apps/Internet/Netscape.desktop",
+	      { "applnk/gnome-help.desktop",
+		"applnk/System/gnome-terminal.desktop",
+		"applnk/Settings/gnomecc.desktop",
+		"applnk/Internet/Netscape.desktop",
 		NULL };
 	int i;
 	char *p;
@@ -978,15 +974,6 @@
 					flags |= MAIN_MENU_SYSTEM_SUB|
 						MAIN_MENU_USER;
 				}
-				/*guess redhat menus*/
-				if(g_file_exists("/etc/X11/wmconfig"))
-					flags |= MAIN_MENU_REDHAT_SUB;
-				/*guess KDE menus */
-				if(g_file_exists(kde_menudir))
-					flags |= MAIN_MENU_KDE_SUB;
-				/*guess debian menus*/
-				if (g_file_exists(DEBIAN_MENUDIR))
-					flags |= MAIN_MENU_DEBIAN_SUB;
 			}
 			if(old_style) {
 				/*this is needed to make panel properly
@@ -997,15 +984,6 @@
 				if(flags&MAIN_MENU_USER &&
 				   flags&MAIN_MENU_USER_SUB)
 					flags &=~ MAIN_MENU_USER;
-				if(flags&MAIN_MENU_REDHAT &&
-				   flags&MAIN_MENU_REDHAT_SUB)
-					flags &=~ MAIN_MENU_REDHAT;
-				if(flags&MAIN_MENU_DEBIAN &&
-				   flags&MAIN_MENU_DEBIAN_SUB)
-					flags &=~ MAIN_MENU_DEBIAN;
-				if(flags&MAIN_MENU_KDE &&
-				   flags&MAIN_MENU_KDE_SUB)
-					flags &=~ MAIN_MENU_KDE;
 				flags |= MAIN_MENU_APPLETS_SUB |
 					MAIN_MENU_PANEL_SUB |
 					MAIN_MENU_DESKTOP;
