diff -ruN gnome-core-1.1.0.orig/desktop-links/Makefile.am gnome-core-1.1.0/desktop-links/Makefile.am
--- gnome-core-1.1.0.orig/desktop-links/Makefile.am	Tue Sep 21 20:01:33 1999
+++ gnome-core-1.1.0/desktop-links/Makefile.am	Tue Jan 25 13:28:52 2000
@@ -1,4 +1,4 @@
-appsdir = $(datadir)/gnome/apps
+appsdir = $(datadir)/applnk
 
 Rootdir           = $(appsdir)
 Applicationsdir   = $(appsdir)/Applications
diff -ruN gnome-core-1.1.0.orig/gmenu/Makefile.am gnome-core-1.1.0/gmenu/Makefile.am
--- gnome-core-1.1.0.orig/gmenu/Makefile.am	Fri Mar 26 23:31:56 1999
+++ gnome-core-1.1.0/gmenu/Makefile.am	Tue Jan 25 13:28:52 2000
@@ -26,5 +26,5 @@
 
 EXTRA_DIST = gmenu.desktop top.xpm unknown.xpm
 
-sysdir = $(datadir)/gnome/apps/Settings
+sysdir = $(datadir)/applnk/Settings
 sys_DATA = gmenu.desktop
diff -ruN gnome-core-1.1.0.orig/gnome-hint/Makefile.am gnome-core-1.1.0/gnome-hint/Makefile.am
--- gnome-core-1.1.0.orig/gnome-hint/Makefile.am	Mon Oct  4 06:44:23 1999
+++ gnome-core-1.1.0/gnome-hint/Makefile.am	Tue Jan 25 13:29:43 2000
@@ -33,5 +33,5 @@
 sys_DATA = gnome-hint-properties.desktop
 
 install-data-local:
-	$(INSTALL_DATA) $(srcdir)/gnome-hint-properties.desktop $(DESTDIR)$(datadir)/gnome/apps/Settings/gnome-hint-properties.desktop
+	$(INSTALL_DATA) $(srcdir)/gnome-hint-properties.desktop $(DESTDIR)$(datadir)/applnk/Settings/gnome-hint-properties.desktop
 
diff -ruN gnome-core-1.1.0.orig/gnome-terminal/Makefile.am gnome-core-1.1.0/gnome-terminal/Makefile.am
--- gnome-core-1.1.0.orig/gnome-terminal/Makefile.am	Thu Oct 21 07:59:40 1999
+++ gnome-core-1.1.0/gnome-terminal/Makefile.am	Tue Jan 25 13:28:52 2000
@@ -28,5 +28,5 @@
 
 gnome_terminal_LDADD = $(LDADD)
 
-utildir = $(datadir)/gnome/apps/Utilities
+utildir = $(datadir)/applnk/Utilities
 util_DATA = gnome-terminal.desktop
diff -ruN gnome-core-1.1.0.orig/gsm/Makefile.am gnome-core-1.1.0/gsm/Makefile.am
--- gnome-core-1.1.0.orig/gsm/Makefile.am	Thu Oct 21 02:29:07 1999
+++ gnome-core-1.1.0/gsm/Makefile.am	Tue Jan 25 13:29:17 2000
@@ -9,7 +9,7 @@
 
 LDADD = $(GNOME_LIBDIR) $(GNOMEUI_LIBS) $(INTLLIBS)
 
-sysdir = $(datadir)/apps/Settings
+sysdir = $(datadir)/applnk/Settings
 
 if SESSION
 noinst_DATA = session.desktop
@@ -34,7 +34,7 @@
 
 default_DATA = default.session default.wm
 
-desktopdir = $(datadir)/gnome/apps/Settings
+desktopdir = $(datadir)/applnk/Settings
 desktop_DATA = save-session.desktop session.desktop
 
 default.session: default.in ../config.status
diff -ruN gnome-core-1.1.0.orig/help-browser/Makefile.am gnome-core-1.1.0/help-browser/Makefile.am
--- gnome-core-1.1.0.orig/help-browser/Makefile.am	Thu Oct 21 07:43:04 1999
+++ gnome-core-1.1.0/help-browser/Makefile.am	Tue Jan 25 13:28:52 2000
@@ -76,5 +76,5 @@
 gnorbadir = $(sysconfdir)/CORBA/servers
 gnorba_DATA = help-browser.gnorba
 
-sysdir = $(datadir)/gnome/apps
+sysdir = $(datadir)/applnk
 sys_DATA = gnome-help.desktop
diff -ruN gnome-core-1.1.0.orig/panel/Makefile.am gnome-core-1.1.0/panel/Makefile.am
--- gnome-core-1.1.0.orig/panel/Makefile.am	Fri Oct 22 21:33:45 1999
+++ gnome-core-1.1.0/panel/Makefile.am	Tue Jan 25 13:31:27 2000
@@ -215,6 +215,6 @@
 sys_DATA = gnome-panel-properties.desktop
 
 install-data-local:
-	$(INSTALL_DATA) $(srcdir)/gnome-panel-properties.desktop $(DESTDIR)$(datadir)/gnome/apps/Settings/gnome-panel-properties.desktop
+	$(INSTALL_DATA) $(srcdir)/gnome-panel-properties.desktop $(DESTDIR)$(datadir)/applnk/Settings/gnome-panel-properties.desktop
 
 $(srcdir)/panel-util.c: $(CORBA_SRCLIST)
diff -ruN gnome-core-1.1.0.orig/panel/menu.c gnome-core-1.1.0/panel/menu.c
--- gnome-core-1.1.0.orig/panel/menu.c	Fri Oct 22 18:39:53 1999
+++ gnome-core-1.1.0/panel/menu.c	Tue Jan 25 13:28:52 2000
@@ -59,7 +59,7 @@
 	/*just load the menus from disk, don't make the widgets
 	  this just reads the .desktops of the top most directory
 	  and a level down*/
-	char *menu = gnome_datadir_file("gnome/apps");
+	char *menu = gnome_datadir_file("applnk");
 	if(menu)
 		fr_read_dir(NULL,menu,NULL,2);
 	g_free(menu);
@@ -67,13 +67,10 @@
 	if(menu)
 		fr_read_dir(NULL,menu,NULL,2);
 	g_free(menu);
-	menu = gnome_util_home_file("apps");
+	menu = gnome_util_home_file("applnk");
 	if(menu)
 		fr_read_dir(NULL,menu,NULL,2);
 	g_free(menu);
-	/*if redhat menus, use the fork version to read*/
-	if(g_file_exists(REDHAT_MENUDIR))
-		create_rh_menu(TRUE);
 }
 
 /*the most important dialog in the whole application*/
@@ -130,7 +127,7 @@
 {
 	char *s;
 	char *p;
-	p = gnome_util_home_file("apps");
+	p = gnome_util_home_file("applnk");
 	s = g_strdup_printf("cp -r -f %s %s",item_loc,p);
 	g_free(p);
 	system(s);
@@ -649,7 +646,7 @@
 						   GTK_SIGNAL_FUNC(add_app_to_personal),
 						   sim->item_loc);
 				/*ummmm slightly ugly but should work 99% of time*/
-				if(strstr(sim->item_loc,"/.gnome/apps/"))
+				if(strstr(sim->item_loc,"/.applnk/"))
 					gtk_widget_set_sensitive(menuitem,FALSE);
 			}
 
@@ -692,7 +689,7 @@
 					   GTK_SIGNAL_FUNC(add_app_to_personal),
 					   sim->mf->menudir);
 			/*ummmm slightly ugly but should work 99% of time*/
-			if(strstr(sim->mf->menudir,"/.gnome/apps"))
+			if(strstr(sim->mf->menudir,"/.applnk"))
 				gtk_widget_set_sensitive(menuitem,FALSE);
 
 			menuitem = gtk_menu_item_new ();
@@ -2068,7 +2065,7 @@
 static GtkWidget *
 create_system_menu(GtkWidget *menu, int fake_submenus, int fake)
 {
-	char *menu_base = gnome_unconditional_datadir_file ("gnome/apps");
+	char *menu_base = gnome_unconditional_datadir_file ("applnk");
 	char *menudir;
 
 	menudir = g_concat_dir_and_file (menu_base, ".");
@@ -2216,7 +2213,7 @@
 				   menuitem);
 	}
 
-	menu = create_user_menu(_("Favorites"),"apps",NULL,
+	menu = create_user_menu(_("Favorites"),"applnk",NULL,
 				"gnome-favorites.png", TRUE, TRUE, TRUE);
 	if(menu) {
 		menuitem = gtk_menu_item_new ();
@@ -3260,7 +3257,7 @@
 		need_separ = TRUE;
 	}
 	if(flags&MAIN_MENU_USER && !(flags&MAIN_MENU_USER_SUB)) {
-		root_menu = create_user_menu(_("Favorites"), "apps",
+		root_menu = create_user_menu(_("Favorites"), "applnk",
 					     root_menu, "gnome-favorites.png",
 					     fake_submenus, FALSE, FALSE);
 		need_separ = TRUE;
@@ -3311,7 +3308,7 @@
 		if(need_separ)
 			add_menu_separator(root_menu);
 		need_separ = FALSE;
-		menu = create_user_menu(_("Favorites"), "apps", NULL,
+		menu = create_user_menu(_("Favorites"), "applnk", NULL,
 					"gnome-favorites.png",
 					fake_submenus, TRUE, TRUE);
 		menuitem = gtk_menu_item_new ();
@@ -3440,7 +3437,7 @@
 	current_panel = PANEL_WIDGET(menu->button->parent);
 
 	if(!menu->menu) {
-		char *menu_base = gnome_unconditional_datadir_file ("gnome/apps");
+		char *menu_base = gnome_unconditional_datadir_file ("applnk");
 		char *this_menu = get_real_menu_path(menu->path,menu_base);
 		GSList *list = g_slist_append(NULL,this_menu);
 		
@@ -3518,7 +3515,7 @@
 	Menu *menu;
 	int main_menu;
 
-	char *menu_base = gnome_unconditional_datadir_file ("gnome/apps");
+	char *menu_base = gnome_unconditional_datadir_file ("applnk");
 	char *this_menu = get_real_menu_path(arguments,menu_base);
 
 	if (!this_menu) {
diff -ruN gnome-core-1.1.0.orig/panel/session.c gnome-core-1.1.0/panel/session.c
--- gnome-core-1.1.0.orig/panel/session.c	Thu Oct 21 04:48:41 1999
+++ gnome-core-1.1.0/panel/session.c	Tue Jan 25 13:30:20 2000
@@ -673,10 +673,10 @@
 load_default_applets1(PanelWidget *panel)
 {
 	char *def_launchers[] =
-	      { "gnome/apps/gnome-help.desktop",
-		"gnome/apps/Utilities/gnome-terminal.desktop",
-		"gnome/apps/Settings/gnomecc.desktop",
-		"gnome/apps/Applications/Netscape.desktop",
+	      { "applnk/gnome-help.desktop",
+		"applnk/Utilities/gnome-terminal.desktop",
+		"applnk/Settings/gnomecc.desktop",
+		"applnk/Applications/Netscape.desktop",
 		NULL };
 	int i;
 	int flags = MAIN_MENU_SYSTEM|MAIN_MENU_USER|
