diff -urN gnome-core-1.3.1.org/core-docs/fdl/C/Makefile.am gnome-core-1.3.1/core-docs/fdl/C/Makefile.am
--- gnome-core-1.3.1.org/core-docs/fdl/C/Makefile.am	Wed Feb 28 15:03:07 2001
+++ gnome-core-1.3.1/core-docs/fdl/C/Makefile.am	Wed Feb 28 16:30:14 2001
@@ -1,6 +1,9 @@
+figs = 
 app = fdl
 lang = C
 omffiles=fdl-C.omf
 include $(top_srcdir)/app-docs.make
 dist-hook: app-dist-hook
 
+topic.dat:
+	echo "index.html	Free Documentation License" > topic.dat
diff -urN gnome-core-1.3.1.org/core-docs/gpl/C/Makefile.am gnome-core-1.3.1/core-docs/gpl/C/Makefile.am
--- gnome-core-1.3.1.org/core-docs/gpl/C/Makefile.am	Wed Feb 28 15:03:07 2001
+++ gnome-core-1.3.1/core-docs/gpl/C/Makefile.am	Wed Feb 28 16:30:47 2001
@@ -1,6 +1,9 @@
+figs =
 app = gpl
 lang = C
 omffiles=gpl-C.omf
 include $(top_srcdir)/app-docs.make
 dist-hook: app-dist-hook
 
+topic.dat:
+	echo "index.html GPL License" > topic.dat       
diff -urN gnome-core-1.3.1.org/core-docs/lgpl/C/Makefile.am gnome-core-1.3.1/core-docs/lgpl/C/Makefile.am
--- gnome-core-1.3.1.org/core-docs/lgpl/C/Makefile.am	Wed Feb 28 15:03:08 2001
+++ gnome-core-1.3.1/core-docs/lgpl/C/Makefile.am	Wed Feb 28 16:31:05 2001
@@ -1,6 +1,9 @@
+figs =
 app = lgpl
 lang = C
 omffiles=lgpl-C.omf
 include $(top_srcdir)/app-docs.make
 dist-hook: app-dist-hook
 
+topic.dat:
+	echo "index.html LGPL License" > topic.dat       
diff -urN gnome-core-1.3.1.org/gnome-terminal/da/Makefile.am gnome-core-1.3.1/gnome-terminal/da/Makefile.am
--- gnome-core-1.3.1.org/gnome-terminal/da/Makefile.am	Wed Feb 28 15:03:06 2001
+++ gnome-core-1.3.1/gnome-terminal/da/Makefile.am	Wed Feb 28 15:41:30 2001
@@ -1,6 +1,7 @@
 figs =
 app = gnome-terminal
 lang = da
+omffiles =
 # Commenting out for now b/c TOC extraction is broken for this SGML file
 # omffiles = gnome-terminal-da.omf
 include $(top_srcdir)/app-docs.make
diff -urN gnome-core-1.3.1.org/gnome-terminal/it/Makefile.am gnome-core-1.3.1/gnome-terminal/it/Makefile.am
--- gnome-core-1.3.1.org/gnome-terminal/it/Makefile.am	Wed Feb 28 15:03:06 2001
+++ gnome-core-1.3.1/gnome-terminal/it/Makefile.am	Wed Feb 28 15:41:59 2001
@@ -1,6 +1,7 @@
 figs =
 app = gnome-terminal
 lang = it
+omffiles =
 # Commenting out for now b/c TOC extraction is broken for this SGML file
 #omffiles = gnome-terminal-it.omf
 include $(top_srcdir)/app-docs.make
diff -urN gnome-core-1.3.1.org/gnome-terminal/no/Makefile.am gnome-core-1.3.1/gnome-terminal/no/Makefile.am
--- gnome-core-1.3.1.org/gnome-terminal/no/Makefile.am	Wed Feb 28 15:03:07 2001
+++ gnome-core-1.3.1/gnome-terminal/no/Makefile.am	Wed Feb 28 15:42:09 2001
@@ -1,6 +1,7 @@
 figs =
 app = gnome-terminal
 lang = no
+omffiles =
 # Commenting out for now b/c TOC extraction is broken for this SGML file
 # omffiles = gnome-terminal-no.omf
 include $(top_srcdir)/app-docs.make
diff -urN gnome-core-1.3.1.org/gsm/help/C/Makefile.am gnome-core-1.3.1/gsm/help/C/Makefile.am
--- gnome-core-1.3.1.org/gsm/help/C/Makefile.am	Wed Feb 28 15:03:00 2001
+++ gnome-core-1.3.1/gsm/help/C/Makefile.am	Wed Feb 28 16:07:32 2001
@@ -14,3 +14,5 @@
 include $(top_srcdir)/app-docs.make
 dist-hook: app-dist-hook
 
+topic.dat:
+	touch topic.dat
diff -urN gnome-core-1.3.1.org/gsm/help/C/topic.dat gnome-core-1.3.1/gsm/help/C/topic.dat
--- gnome-core-1.3.1.org/gsm/help/C/topic.dat	Thu Jan  1 01:00:00 1970
+++ gnome-core-1.3.1/gsm/help/C/topic.dat	Wed Feb 28 16:00:58 2001
@@ -0,0 +1 @@
+index.html GNOME session manager guide. 
diff -urN gnome-core-1.3.1.org/smproxy/Makefile.am gnome-core-1.3.1/smproxy/Makefile.am
--- gnome-core-1.3.1.org/smproxy/Makefile.am	Wed Feb 28 15:03:05 2001
+++ gnome-core-1.3.1/smproxy/Makefile.am	Wed Feb 28 16:15:03 2001
@@ -13,6 +13,4 @@
 
 gnome_smproxy_SOURCES = save.c smproxy.h smproxy.c
 
-man1_MANS = smproxy.man
-
 EXTRA_DIST = README.gnome
