--- gnome-core-1.4.0.4/configure.in~	Tue Sep 18 06:51:32 2001
+++ gnome-core-1.4.0.4/configure.in	Tue Sep 18 06:56:14 2001
@@ -64,9 +64,6 @@
 CFLAGS=$CFLAGS_save
 LIBS=$LIBS_save
 
-dnl utility conditional
-AM_CONDITIONAL(FALSE, test "x" = "y")
-
 ALL_LINGUAS="az bg ca cs da de el en_GB es et eu fi fr ga gl hu it ja ko lt lv nl nn no pl pt pt_BR ro ru sk sl sp sr sv ta tr uk wa zh_TW.Big5 zh_CN.GB2312"
 AM_GNU_GETTEXT
 # AM_GNOME_GETTEXT above substs $DATADIRNAME
