--- gnome-core-1.0.41/desktop-links/Makefile.am	Fri Feb 26 13:52:03 1999
+++ gnome-core-1.0.41.new/desktop-links/Makefile.am	Wed Sep 29 05:41:45 1999
@@ -1,4 +1,4 @@
-appsdir = $(datadir)/gnome/apps
+appsdir = $(datadir)/applnk
 
 Rootdir           = $(appsdir)
 Applicationsdir   = $(appsdir)/Applications
--- gnome-core-1.0.41/gmenu/Makefile.am	Fri Mar 26 23:31:56 1999
+++ gnome-core-1.0.41.new/gmenu/Makefile.am	Wed Sep 29 05:40:36 1999
@@ -26,5 +26,5 @@
 
 EXTRA_DIST = gmenu.desktop top.xpm unknown.xpm
 
-sysdir = $(datadir)/gnome/apps/Settings
+sysdir = $(datadir)/applnk/Settings
 sys_DATA = gmenu.desktop
--- gnome-core-1.0.41/gnome-terminal/Makefile.am	Wed Aug 18 01:48:11 1999
+++ gnome-core-1.0.41.new/gnome-terminal/Makefile.am	Wed Sep 29 05:42:17 1999
@@ -28,5 +28,5 @@
 
 gnome_terminal_LDADD = $(LDADD)
 
-utildir = $(datadir)/gnome/apps/Utilities
+utildir = $(datadir)/applnk/Utilities
 util_DATA = gnome-terminal.desktop
--- gnome-core-1.0.41/gsm/Makefile.am	Tue Sep 21 07:40:54 1999
+++ gnome-core-1.0.41.new/gsm/Makefile.am	Wed Sep 29 05:43:03 1999
@@ -9,7 +9,7 @@
 
 LDADD = $(GNOME_LIBDIR) $(GNOMEUI_LIBS) $(INTLLIBS)
 
-sysdir = $(datadir)/apps/Settings
+sysdir = $(datadir)/applnk/Settings
 
 if SESSION
 bin_PROGRAMS = gnome-session save-session gnome-login-check session-properties
--- gnome-core-1.0.41/help-browser/Makefile.am	Fri Sep 24 02:05:14 1999
+++ gnome-core-1.0.41.new/help-browser/Makefile.am	Wed Sep 29 05:43:43 1999
@@ -76,5 +76,5 @@
 gnorbadir = $(sysconfdir)/CORBA/servers
 gnorba_DATA = help-browser.gnorba
 
-sysdir = $(datadir)/gnome/apps
+sysdir = $(datadir)/applnk
 sys_DATA = gnome-help.desktop
--- gnome-core-1.0.53/panel/menu.c~	Tue Oct  5 23:34:05 1999
+++ gnome-core-1.0.53/panel/menu.c	Thu Oct 14 15:09:40 1999
@@ -506,7 +506,7 @@
 	/*just load the menus from disk, don't make the widgets
 	  this just reads the .desktops of the top most directory
 	  and a level down*/
-	char *menu = gnome_datadir_file("gnome/apps");
+	char *menu = gnome_datadir_file("applnk");
 	if(menu)
 		fr_read_dir(NULL,menu,NULL,2);
 	g_free(menu);
@@ -514,13 +514,10 @@
 	if(menu)
 		fr_read_dir(NULL,menu,NULL,2);
 	g_free(menu);
-	menu = gnome_util_home_file("apps");
+	menu = gnome_util_home_file("applnk");
 	if(menu)
 		fr_read_dir(NULL,menu,NULL,2);
 	g_free(menu);
-	/*if redhat menus, use the fork version to read*/
-	if(g_file_exists("/etc/X11/wmconfig"))
-		create_rh_menu(TRUE);
 }
 
 /*the most important dialog in the whole application*/
@@ -575,7 +572,7 @@
 {
 	char *s;
 	char *p;
-	p = gnome_util_home_file("apps");
+	p = gnome_util_home_file("applnk");
 	s = g_strdup_printf("cp -r -f %s %s",item_loc,p);
 	g_free(p);
 	system(s);
@@ -896,7 +893,7 @@
 					   GTK_SIGNAL_FUNC(add_app_to_personal),
 					   sim->item_loc);
 			/*ummmm slightly ugly but should work 99% of time*/
-			if(strstr(sim->item_loc,"/.gnome/apps/"))
+			if(strstr(sim->item_loc,"/.applnk/"))
 				gtk_widget_set_sensitive(menuitem,FALSE);
 		} else {
 			menuitem = gtk_menu_item_new ();
@@ -922,7 +919,7 @@
 					   GTK_SIGNAL_FUNC(add_app_to_personal),
 					   sim->mf->menudir);
 			/*ummmm slightly ugly but should work 99% of time*/
-			if(strstr(sim->mf->menudir,"/.gnome/apps"))
+			if(strstr(sim->mf->menudir,"/.applnk"))
 				gtk_widget_set_sensitive(menuitem,FALSE);
 		}
 
@@ -2081,7 +2078,7 @@
 static GtkWidget *
 create_system_menu(GtkWidget *menu, int fake_submenus, int fake)
 {
-	char *menu_base = gnome_unconditional_datadir_file ("gnome/apps");
+	char *menu_base = gnome_unconditional_datadir_file ("applnk");
 	char *menudir, *pixmap;
 	
 	menudir = g_concat_dir_and_file (menu_base, ".");
@@ -2195,7 +2192,7 @@
 	}
 
 	pixmap = gnome_pixmap_file ("gnome-favorites.png");
-	menu = create_user_menu(_("User menus"),"apps",NULL,pixmap,TRUE,TRUE,TRUE);
+	menu = create_user_menu(_("User menus"),"applnk",NULL,pixmap,TRUE,TRUE,TRUE);
 	g_free (pixmap);
 	if(menu) {
 		menuitem = gtk_menu_item_new ();
@@ -2986,7 +2983,7 @@
 	}
 	if(flags&MAIN_MENU_USER && !(flags&MAIN_MENU_USER_SUB)) {
 		pixmap = gnome_pixmap_file ("gnome-favorites.png");
-		root_menu = create_user_menu(_("User menus"), "apps", 
+		root_menu = create_user_menu(_("User menus"), "applnk", 
 					     root_menu, pixmap,
 					     fake_submenus, FALSE,
 					     FALSE);
@@ -3036,7 +3033,7 @@
 			add_menu_separator(root_menu);
 		need_separ = FALSE;
 		pixmap = gnome_pixmap_file ("gnome-favorites.png");
-		menu = create_user_menu(_("User menus"), "apps", 
+		menu = create_user_menu(_("User menus"), "applnk", 
 					NULL, pixmap,
 					fake_submenus, TRUE, TRUE);
 		g_free (pixmap);
@@ -3148,7 +3145,7 @@
 	current_panel = PANEL_WIDGET(menu->button->parent);
 
 	if(!menu->menu) {
-		char *menu_base = gnome_unconditional_datadir_file ("gnome/apps");
+		char *menu_base = gnome_unconditional_datadir_file ("applnk");
 		char *this_menu = get_real_menu_path(menu->path,menu_base);
 		GSList *list = g_slist_append(NULL,this_menu);
 		
@@ -3257,7 +3254,7 @@
 	Menu *menu;
 	int main_menu;
 
-	char *menu_base = gnome_unconditional_datadir_file ("gnome/apps");
+	char *menu_base = gnome_unconditional_datadir_file ("applnk");
 	char *this_menu = get_real_menu_path(arguments,menu_base);
 
 	if (!this_menu) {
@@ -3377,7 +3374,7 @@
 	menu->menu = NULL;
 
 	{
-		char *menu_base = gnome_unconditional_datadir_file ("gnome/apps");
+		char *menu_base = gnome_unconditional_datadir_file ("applnk");
 		char *this_menu = get_real_menu_path(menu->path,menu_base);
 		GSList *list = g_slist_append(NULL,this_menu);
 		char *pixmap_name = get_pixmap(this_menu,
