diff -urN gnome-core-1.3.1.org/desktop-links/Makefile.am gnome-core-1.3.1/desktop-links/Makefile.am
--- gnome-core-1.3.1.org/desktop-links/Makefile.am	Wed Feb 28 16:53:09 2001
+++ gnome-core-1.3.1/desktop-links/Makefile.am	Wed Feb 28 16:53:33 2001
@@ -1,4 +1,4 @@
-appsdir = $(datadir)/gnome/apps
+appsdir = $(datadir)/applnk
 
 Rootdir           = $(appsdir)
 Applicationsdir   = $(appsdir)/Applications
diff -urN gnome-core-1.3.1.org/gmenu/Makefile.am gnome-core-1.3.1/gmenu/Makefile.am
--- gnome-core-1.3.1.org/gmenu/Makefile.am	Wed Feb 28 16:53:00 2001
+++ gnome-core-1.3.1/gmenu/Makefile.am	Wed Feb 28 16:53:33 2001
@@ -27,5 +27,5 @@
 
 EXTRA_DIST = gmenu.desktop top.xpm unknown.xpm
 
-sysdir = $(datadir)/gnome/apps/Settings
+sysdir = $(datadir)/applnk/Settings/GNOME
 sys_DATA = gmenu.desktop
diff -urN gnome-core-1.3.1.org/gmenu/main.c gnome-core-1.3.1/gmenu/main.c
--- gnome-core-1.3.1.org/gmenu/main.c	Wed Feb 28 16:53:00 2001
+++ gnome-core-1.3.1/gmenu/main.c	Wed Feb 28 16:53:33 2001
@@ -279,7 +279,7 @@
 	gnome_init ("GNOME menu editor", VERSION, argc, argv);
 	gnome_window_icon_set_default_from_file (GNOME_ICONDIR"/gnome-gmenu.png");
 
-	system_apps_dir = gnome_datadir_file("gnome/apps");
+	system_apps_dir = gnome_datadir_file("applnk");
 	system_applets_dir = gnome_datadir_file("applets");
 	system_pixmap_dir = gnome_datadir_file("pixmaps");
 	if (!system_apps_dir || !system_pixmap_dir || !system_applets_dir)
@@ -301,7 +301,7 @@
 		}
 	}
 
-	user_apps_dir = check_for_dir(gnome_util_home_file("apps"));
+	user_apps_dir = check_for_dir(gnome_util_home_file("applnk"));
 	user_pixmap_dir = check_for_dir(gnome_util_home_file("pixmaps"));
 
 	app = gnome_app_new ("gmenu",_("GNOME menu editor"));
diff -urN gnome-core-1.3.1.org/gnome-hint/Makefile.am gnome-core-1.3.1/gnome-hint/Makefile.am
--- gnome-core-1.3.1.org/gnome-hint/Makefile.am	Wed Feb 28 16:53:03 2001
+++ gnome-core-1.3.1/gnome-hint/Makefile.am	Wed Feb 28 16:53:33 2001
@@ -23,7 +23,7 @@
 	$(GNOME_CAPPLET_LIBS)	\
 	$(INTLLIBS)
 
-utildir = $(datadir)/gnome/apps/Utilities
+utildir = $(datadir)/applnk/Utilities
 util_DATA = gnome-hint.desktop
 
 EXTRA_DIST = \
@@ -37,7 +37,6 @@
 pixmapsdir = $(datadir)/pixmaps
 pixmaps_DATA = gnome-hint.png
 
-install-data-local:
-	$(mkinstalldirs) $(DESTDIR)$(datadir)/gnome/apps/Settings/Session/
-	$(INSTALL_DATA) $(srcdir)/gnome-hint-properties.desktop $(DESTDIR)$(datadir)/gnome/apps/Settings/Session/gnome-hint-properties.desktop
+desktopdir = $(datadir)/applnk/Settings/GNOME/Session
+desktop_DATA = gnome-hint-properties.desktop
 
diff -urN gnome-core-1.3.1.org/gnome-terminal/Makefile.am gnome-core-1.3.1/gnome-terminal/Makefile.am
--- gnome-core-1.3.1.org/gnome-terminal/Makefile.am	Wed Feb 28 16:53:04 2001
+++ gnome-core-1.3.1/gnome-terminal/Makefile.am	Wed Feb 28 16:53:33 2001
@@ -42,7 +42,7 @@
 gnorbadir = $(sysconfdir)/CORBA/servers
 gnorba_DATA = gnome-terminal.gnorba
 
-utildir = $(datadir)/gnome/apps/System
+utildir = $(datadir)/applnk/System
 util_DATA = gnome-terminal.desktop
 
 gladedir   = $(datadir)/gnome-terminal/glade
diff -urN gnome-core-1.3.1.org/gsm/Makefile.am gnome-core-1.3.1/gsm/Makefile.am
--- gnome-core-1.3.1.org/gsm/Makefile.am	Wed Feb 28 16:53:00 2001
+++ gnome-core-1.3.1/gsm/Makefile.am	Wed Feb 28 16:53:33 2001
@@ -11,7 +11,7 @@
 
 LDADD = $(GNOME_LIBDIR) $(GNOMEUI_LIBS) $(INTLLIBS)
 
-sysdir = $(datadir)/apps/Settings
+sysdir = $(datadir)/applnk/Settings/GNOME
 
 if SESSION
 noinst_DATA = session.desktop
@@ -48,7 +48,7 @@
 
 default_DATA = default.session default.wm
 
-desktopdir = $(datadir)/gnome/apps/Settings/Session/
+desktopdir = $(datadir)/applnk/GNOME/Settings/Session/
 desktop_DATA = save-session.desktop session.desktop
 
 sysdir = $(datadir)/control-center/Session
@@ -57,7 +57,7 @@
 sys_DATA = session-properties.desktop
 endif
 
-settingsdir = $(datadir)/gnome/apps/Settings/Session
+settingsdir = $(datadir)/applnk/Settings/GNOME/Session
 
 if SESSION
 settings_DATA = session-properties.desktop
diff -urN gnome-core-1.3.1.org/help-browser/Makefile.am gnome-core-1.3.1/help-browser/Makefile.am
--- gnome-core-1.3.1.org/help-browser/Makefile.am	Wed Feb 28 16:53:09 2001
+++ gnome-core-1.3.1/help-browser/Makefile.am	Wed Feb 28 16:53:33 2001
@@ -89,7 +89,7 @@
 gnorbadir = $(sysconfdir)/CORBA/servers
 gnorba_DATA = help-browser.gnorba
 
-sysdir = $(datadir)/gnome/apps
+sysdir = $(datadir)/applnk
 sys_DATA = gnome-help.desktop
 
 install-exec-local:
diff -urN gnome-core-1.3.1.org/panel/Makefile.am gnome-core-1.3.1/panel/Makefile.am
--- gnome-core-1.3.1.org/panel/Makefile.am	Wed Feb 28 16:53:01 2001
+++ gnome-core-1.3.1/panel/Makefile.am	Wed Feb 28 16:53:33 2001
@@ -108,14 +108,10 @@
 	panel_config.c  	\
 	panel_config.h		\
 	panel_config_global.h  	\
-	distribution.c		\
-	distribution.h		\
 	menu.c  		\
 	menu.h  		\
 	menu-fentry.c  		\
 	menu-fentry.h  		\
-	menu-rh.c  		\
-	menu-rh.h  		\
 	menu-properties.c	\
 	menu-properties.h	\
 	menu-util.c		\
@@ -258,7 +254,7 @@
 sysdir = $(datadir)/control-center/Desktop/
 sys_DATA = gnome-panel-properties.desktop
 
-deskdir = $(datadir)/gnome/apps/Settings/Desktop/
+deskdir = $(datadir)/applnk/Settings/GNOME/Desktop/
 desk_DATA = gnome-panel-properties.desktop
 
 $(srcdir)/panel-util.c: $(CORBA_SRCLIST)
diff -urN gnome-core-1.3.1.org/panel/foobar-widget.c gnome-core-1.3.1/panel/foobar-widget.c
--- gnome-core-1.3.1.org/panel/foobar-widget.c	Wed Feb 28 16:53:02 2001
+++ gnome-core-1.3.1/panel/foobar-widget.c	Wed Feb 28 16:53:33 2001
@@ -976,10 +976,10 @@
 
 	foo->favorites =
 		append_folder_menu(menu_bar, _("Favorites"), NULL,
-				   FALSE, "apps");
+				   FALSE, "applnk");
 	foo->settings =
 		append_folder_menu(menu_bar, _("Settings"),  NULL, TRUE,
-			           "gnome/apps/Settings");
+			           "applnk/Settings/GNOME");
 	append_desktop_menu (menu_bar);
 
 	gtk_box_pack_start (GTK_BOX (foo->hbox), menu_bar, FALSE, FALSE, 0);
diff -urN gnome-core-1.3.1.org/panel/gnome-panel-properties.c gnome-core-1.3.1/panel/gnome-panel-properties.c
--- gnome-core-1.3.1.org/panel/gnome-panel-properties.c	Wed Feb 28 16:53:01 2001
+++ gnome-core-1.3.1/panel/gnome-panel-properties.c	Wed Feb 28 16:53:33 2001
@@ -118,8 +118,6 @@
 	{ MAIN_MENU_SYSTEM,       MAIN_MENU_SYSTEM_SUB,        N_("Programs: ") },
 	{ MAIN_MENU_USER,         MAIN_MENU_USER_SUB,          N_("Favorites: ") },
 	{ MAIN_MENU_APPLETS,      MAIN_MENU_APPLETS_SUB,       N_("Applets: ") },
-	{ MAIN_MENU_DISTRIBUTION, MAIN_MENU_DISTRIBUTION_SUB,  N_("Distribution: ") },
-	{ MAIN_MENU_KDE,          MAIN_MENU_KDE_SUB,           N_("KDE: ") },
 	{ MAIN_MENU_PANEL,        MAIN_MENU_PANEL_SUB,         N_("Panel: ") },
 	{ MAIN_MENU_DESKTOP,      MAIN_MENU_DESKTOP_SUB,       N_("Desktop: ") },
 	{ 0 }
@@ -1280,8 +1278,7 @@
 
 	g_string_sprintf (buf, "menu_flags=%d", 
 			  (int)(MAIN_MENU_SYSTEM_SUB | MAIN_MENU_USER_SUB|
-				MAIN_MENU_APPLETS_SUB | MAIN_MENU_DISTRIBUTION_SUB|
-				MAIN_MENU_KDE_SUB | MAIN_MENU_PANEL | MAIN_MENU_DESKTOP));
+				MAIN_MENU_APPLETS_SUB | MAIN_MENU_PANEL | MAIN_MENU_DESKTOP));
 	global_config.menu_flags = gnome_config_get_int(buf->str);
 
 	global_config.keys_enabled =
diff -urN gnome-core-1.3.1.org/panel/launcher.c gnome-core-1.3.1/panel/launcher.c
--- gnome-core-1.3.1.org/panel/launcher.c	Wed Feb 28 16:53:01 2001
+++ gnome-core-1.3.1/panel/launcher.c	Wed Feb 28 16:53:33 2001
@@ -319,7 +319,7 @@
 			else
 				extension = ".desktop";
 
-			apps_par = g_strconcat ("apps/", parameters,
+			apps_par = g_strconcat ("applnk/", parameters,
 						extension, NULL);
 			entry = gnome_datadir_file (apps_par);
 			g_free (apps_par);
diff -urN gnome-core-1.3.1.org/panel/main.c gnome-core-1.3.1/panel/main.c
--- gnome-core-1.3.1.org/panel/main.c	Wed Feb 28 16:53:01 2001
+++ gnome-core-1.3.1/panel/main.c	Wed Feb 28 17:58:45 2001
@@ -43,9 +43,11 @@
 
 GnomeClient *client = NULL;
 
+#if 0
 char *kde_menudir = NULL;
 char *kde_icondir = NULL;
 char *kde_mini_icondir = NULL;
+#endif
 
 char *merge_merge_dir = NULL;
 int merge_main_dir_len = 0;
@@ -100,6 +102,7 @@
 	return TRUE;
 }
 
+#if 0
 static void
 find_kde_directory(void)
 {
@@ -169,6 +172,7 @@
 		merge_merge_dir = tmp;
 	}
 }
+#endif
 
 static void
 setup_visuals (void)
@@ -223,9 +227,11 @@
 	}
 	}
 
+#if 0
 	setup_merge_directory();
 
 	find_kde_directory();
+#endif
 
 	client = gnome_master_client ();
 
diff -urN gnome-core-1.3.1.org/panel/menu-properties.c gnome-core-1.3.1/panel/menu-properties.c
--- gnome-core-1.3.1.org/panel/menu-properties.c	Wed Feb 28 16:53:03 2001
+++ gnome-core-1.3.1/panel/menu-properties.c	Wed Feb 28 16:53:33 2001
@@ -40,10 +40,12 @@
 	GtkWidget *user_sub;
 	GtkWidget *applets;
 	GtkWidget *applets_sub;
+#if 0
 	GtkWidget *distribution;
 	GtkWidget *distribution_sub;
  	GtkWidget *kde;
  	GtkWidget *kde_sub;
+#endif
 	GtkWidget *panel;
 	GtkWidget *panel_sub;
 	GtkWidget *desktop;
@@ -66,7 +68,7 @@
 		arguments = ".";
 
 	if(strcmp(arguments, ".") == 0)
-		this_menu = gnome_unconditional_datadir_file ("gnome/apps");
+		this_menu = gnome_unconditional_datadir_file ("applnk");
 	else if (*arguments == '/')
 		this_menu = g_strdup (arguments);
 	else if (*arguments == '~')
@@ -233,6 +235,7 @@
 	else if (GTK_TOGGLE_BUTTON (menu->dialog_info->applets)->active)
 		menu->main_menu_flags |= MAIN_MENU_APPLETS;
 
+#if 0
 	if(GTK_TOGGLE_BUTTON(menu->dialog_info->distribution_sub)->active)
 		menu->main_menu_flags |= MAIN_MENU_DISTRIBUTION_SUB;
 	else if (GTK_TOGGLE_BUTTON (menu->dialog_info->distribution)->active)
@@ -242,6 +245,7 @@
 		menu->main_menu_flags |= MAIN_MENU_KDE_SUB;
 	else if (GTK_TOGGLE_BUTTON (menu->dialog_info->kde)->active)
 		menu->main_menu_flags |= MAIN_MENU_KDE;
+#endif
 
 	if(GTK_TOGGLE_BUTTON(menu->dialog_info->panel_sub)->active)
 		menu->main_menu_flags |= MAIN_MENU_PANEL_SUB;
@@ -548,6 +552,7 @@
 			      &menu->dialog_info->applets_sub,
 			      menu->main_menu_flags & MAIN_MENU_APPLETS,
 			      menu->main_menu_flags & MAIN_MENU_APPLETS_SUB);
+#if 0
 	add_menu_type_options(menu,
 			      GTK_OBJECT(dialog), GTK_TABLE(table),3,
 			      _("Distribution menu (if found): "),
@@ -562,6 +567,7 @@
 			      &menu->dialog_info->kde_sub,
  			      menu->main_menu_flags & MAIN_MENU_KDE,
  			      menu->main_menu_flags & MAIN_MENU_KDE_SUB);
+#endif
 	add_menu_type_options(menu,
 			      GTK_OBJECT(dialog), GTK_TABLE(table),6,
 			      _("Panel menu: "),
diff -urN gnome-core-1.3.1.org/panel/menu-util.c gnome-core-1.3.1/panel/menu-util.c
--- gnome-core-1.3.1.org/panel/menu-util.c	Wed Feb 28 16:53:01 2001
+++ gnome-core-1.3.1/panel/menu-util.c	Wed Feb 28 16:53:33 2001
@@ -13,9 +13,11 @@
 
 /*#define PANEL_DEBUG 1*/
 
+#if 0
 extern char *kde_menudir;
 extern char *kde_icondir;
 extern char *kde_mini_icondir;
+#endif
 
 GtkWidget *
 add_menu_separator (GtkWidget *menu)
@@ -98,12 +100,13 @@
 int
 get_default_menu_flags (void)
 {
-	DistributionType distribution = get_distribution_type();
+/*	DistributionType distribution = get_distribution_type(); */
 
 	int flags = MAIN_MENU_SYSTEM_SUB | MAIN_MENU_USER_SUB |
 		MAIN_MENU_APPLETS_SUB | MAIN_MENU_PANEL_SUB |
 		MAIN_MENU_DESKTOP;
-	
+
+#if 0
 	/*guess distribution menus*/
 	if(distribution != DISTRIBUTION_UNKNOWN)
 		flags |= MAIN_MENU_DISTRIBUTION_SUB;
@@ -111,6 +114,7 @@
 	/*guess KDE menus*/
 	if (panel_file_exists (kde_menudir))
 		flags |= MAIN_MENU_KDE_SUB;
+#endif
 
 	return flags;
 }
diff -urN gnome-core-1.3.1.org/panel/menu.c gnome-core-1.3.1/panel/menu.c
--- gnome-core-1.3.1.org/panel/menu.c	Wed Feb 28 16:53:02 2001
+++ gnome-core-1.3.1/panel/menu.c	Wed Feb 28 17:55:02 2001
@@ -58,9 +58,11 @@
 
 extern int base_panels;
 
+#if 0
 extern char *kde_menudir;
 extern char *kde_icondir;
 extern char *kde_mini_icondir;
+#endif
 
 extern GtkTooltips *panel_tooltips;
 
@@ -138,13 +140,13 @@
 void
 init_menus (void)
 {
-	const DistributionInfo *distribution_info = get_distribution_info ();
+/*	const DistributionInfo *distribution_info = get_distribution_info (); */
 	char *menu;
 
 	/*just load the menus from disk, don't make the widgets
 	  this just reads the .desktops of the top most directory
 	  and a level down*/
-	menu = gnome_datadir_file ("gnome/apps");
+	menu = gnome_datadir_file ("applnk");
 	if (menu != NULL)
 		fr_read_dir (NULL, menu, NULL, NULL, 2);
 	g_free (menu);
@@ -154,14 +156,16 @@
 		fr_read_dir (NULL, menu, NULL, NULL, 2);
 	g_free (menu);
 
-	menu = gnome_util_home_file ("apps");
+	menu = gnome_util_home_file ("applnk");
 	if (menu != NULL)
 		fr_read_dir (NULL, menu, NULL, NULL, 2);
 	g_free (menu);
 
+#if 0
 	if (distribution_info != NULL &&
 	    distribution_info->menu_init_func != NULL)
 		distribution_info->menu_init_func ();
+#endif
 }
 
 /*the most important dialog in the whole application*/
@@ -330,7 +334,7 @@
 {
 	char *to;
 
-	to = gnome_util_home_file ("apps");
+	to = gnome_util_home_file ("applnk");
 
 	if (g_file_test (item_loc, G_FILE_TEST_ISDIR)) {
 		FileRec *fr = fr_get_dir (item_loc);
@@ -1098,11 +1102,12 @@
 {
 	char *menudir = data;
 	PanelWidget *panel;
-	DistributionType distribution = get_distribution_type ();
+/*	DistributionType distribution = get_distribution_type (); */
 	int flags = MAIN_MENU_SYSTEM_SUB | MAIN_MENU_USER_SUB |
 		MAIN_MENU_APPLETS_SUB | MAIN_MENU_PANEL_SUB |
 		MAIN_MENU_DESKTOP_SUB;
 	
+#if 0
 	/*guess distribution menus*/
 	if (distribution != DISTRIBUTION_UNKNOWN)
 		flags |= MAIN_MENU_DISTRIBUTION_SUB;
@@ -1110,6 +1115,7 @@
 	/*guess KDE menus*/
 	if(panel_file_exists(kde_menudir))
 		flags |= MAIN_MENU_KDE_SUB;
+#endif
 
 	panel = get_panel_from_menu_data (widget);
 
@@ -1244,9 +1250,11 @@
 {
 	errno = 0;
 	if (sim->item_loc != NULL &&
+#if 0
 	    /*A HACK: but it works, don't have it edittable if it's redhat
 	      menus as they are auto generated!*/
 	    strstr (sim->item_loc,"/.gnome/apps-redhat/") == NULL &&
+#endif
 	    /*if it's a kdelnk file, don't let it be editted*/
 	    ! is_ext (sim->item_loc, ".kdelnk") &&
 	    access (sim->item_loc, W_OK) == 0) {
@@ -1260,10 +1268,13 @@
 		   sim->mf != NULL) {
 		/*the dentry isn't there, check if we can write the
 		  directory*/
+#if 0
 		if (access (sim->mf->menudir, W_OK) == 0 &&
 		   /*A HACK: but it works, don't have it edittable if it's redhat
 		     menus as they are auto generated!*/
 		   strstr (sim->mf->menudir, ".gnome/apps-redhat/") == NULL)
+#endif
+		 if (access (sim->mf->menudir, W_OK) == 0)
 			return TRUE;
 	}
 	return FALSE;
@@ -1298,12 +1309,14 @@
 	g_return_if_fail (sim->item_loc != NULL);
 
 	dentry = gnome_desktop_entry_load_unconditional (sim->item_loc);
+#if 0
 	/* We'll screw up a KDE menu entry if we edit it */
 	if (dentry != NULL &&
 	    dentry->is_kde) {
 		gnome_desktop_entry_free (dentry);
 		return;
 	}
+#endif
 
 	/* watch the enum at the top of the file */
 	dialog = gnome_dialog_new (_("Desktop entry properties"),
@@ -1408,12 +1421,14 @@
 	GList *types = NULL;
 
 	dentry = gnome_desktop_entry_load_unconditional(dirfile);
+#if 0
 	/* We'll screw up a KDE menu entry if we edit it */
 	if (dentry != NULL &&
 	    dentry->is_kde) {
 		gnome_desktop_entry_free (dentry);
 		return;
 	}
+#endif
 
 	/* watch the enum at the top of the file */
 	dialog = gnome_dialog_new (_("Desktop entry properties"),
@@ -1567,7 +1582,7 @@
 						   GTK_SIGNAL_FUNC(add_app_to_personal),
 						   (gpointer)sim->item_loc);
 				/*ummmm slightly ugly but should work 99% of time*/
-				if (strstr(sim->item_loc,"/.gnome/apps/") != NULL)
+				if (strstr(sim->item_loc,"/.applnk/") != NULL)
 					gtk_widget_set_sensitive(menuitem,FALSE);
 			}
 
@@ -1632,7 +1647,7 @@
 					   GTK_SIGNAL_FUNC(add_app_to_personal),
 					   sim->mf->menudir);
 			/*ummmm slightly ugly but should work 99% of time*/
-			if(strstr(sim->mf->menudir, "/.gnome/apps") != NULL)
+			if(strstr(sim->mf->menudir, "/.applnk") != NULL)
 				gtk_widget_set_sensitive(menuitem, FALSE);
 
 			menuitem = gtk_menu_item_new ();
@@ -3376,7 +3391,7 @@
 		    gboolean title,
 		    gboolean launcher_add)
 {
-	char *menudir = gnome_datadir_file ("gnome/apps");
+	char *menudir = gnome_datadir_file ("applnk");
 
 	if (menudir &&
 	    g_file_test (menudir, G_FILE_TEST_ISDIR)) {
@@ -3441,6 +3456,7 @@
 	return menu;
 }
 
+#if 0
 static GtkWidget *
 create_distribution_menu (GtkWidget *menu,
 			  gboolean fake_submenus,
@@ -3512,6 +3528,7 @@
 	g_free (pixmap_name);
 	return menu;
 }
+#endif
 
 static GtkWidget *
 create_add_launcher_menu (GtkWidget *menu, gboolean fake_submenus)
@@ -3528,10 +3545,11 @@
 			     GINT_TO_POINTER (1));
 
 	create_system_menu (menu, fake_submenus, FALSE, TRUE, TRUE);
-	create_user_menu (_("Favorites"), "apps",
+	create_user_menu (_("Favorites"), "applnk",
 			  menu, "gnome-favorites.png",
 			  fake_submenus, FALSE, FALSE, TRUE, TRUE);
 
+#if 0
 	add_menu_separator (menu);
 
 	add_distribution_submenu (menu, fake_submenus,
@@ -3540,6 +3558,7 @@
 		add_kde_submenu (menu, fake_submenus,
 				 TRUE /*launcher_add */);
 	}
+#endif
 
 	return menu;
 }
@@ -4505,8 +4524,8 @@
 	gtk_menu_append (GTK_MENU (submenu), submenuitem);
 	gtk_signal_connect(GTK_OBJECT(submenuitem), "activate",
 			   GTK_SIGNAL_FUNC(add_menu_to_panel),
-			   "gnome/apps/");
-	setup_internal_applet_drag(submenuitem, "MENU:gnome/apps/");
+			   "applnk");
+	setup_internal_applet_drag(submenuitem, "MENU:gpplnk/");
 
 	submenuitem = gtk_menu_item_new ();
 	gtk_widget_lock_accelerators (menuitem);
@@ -4515,8 +4534,8 @@
 	gtk_menu_append (GTK_MENU (submenu), submenuitem);
 	gtk_signal_connect(GTK_OBJECT(submenuitem), "activate",
 			   GTK_SIGNAL_FUNC(add_menu_to_panel),
-			   "~/.gnome/apps/");
-	setup_internal_applet_drag(submenuitem, "MENU:~/.gnome/apps/");
+			   "~/.applnk/");
+	setup_internal_applet_drag(submenuitem, "MENU:~/.applnk/");
 
 	menuitem = gtk_menu_item_new ();
 	gtk_widget_lock_accelerators (menuitem);
@@ -4986,6 +5005,7 @@
 	return menu;
 }
 
+#if 0
 static void
 add_distribution_submenu (GtkWidget *root_menu, gboolean fake_submenus,
 			  gboolean launcher_add)
@@ -5048,6 +5068,7 @@
 	gtk_signal_connect(GTK_OBJECT(menu),"show",
 			   GTK_SIGNAL_FUNC(submenu_to_display), NULL);
 }
+#endif
 
 GtkWidget *
 create_root_menu (GtkWidget *root_menu,
@@ -5062,13 +5083,11 @@
 
 	gboolean has_inline = (flags & (MAIN_MENU_SYSTEM |
 					MAIN_MENU_USER |
-					MAIN_MENU_APPLETS |
-					MAIN_MENU_KDE));
+					MAIN_MENU_APPLETS));
 
 	gboolean has_subs = (flags & (MAIN_MENU_SYSTEM_SUB |
 				      MAIN_MENU_USER_SUB |
-				      MAIN_MENU_APPLETS_SUB |
-				      MAIN_MENU_KDE_SUB));
+				      MAIN_MENU_APPLETS_SUB));
 
 	gboolean has_inline2 = (flags & (MAIN_MENU_DESKTOP |
 					 MAIN_MENU_PANEL));
@@ -5078,12 +5097,14 @@
 	IconSize size = global_config.use_large_icons 
 		? MEDIUM_ICON_SIZE : SMALL_ICON_SIZE;
 
+#if 0
 	const DistributionInfo *distribution_info = get_distribution_info ();
 
 	if (distribution_info != NULL) {
 		has_inline |= (flags & (MAIN_MENU_DISTRIBUTION));
 		has_subs |= (flags & (MAIN_MENU_DISTRIBUTION_SUB));
 	}
+#endif
 
 
 	if(!root_menu)
@@ -5101,12 +5122,13 @@
 		create_system_menu(root_menu, fake_submenus,
 				   FALSE, title, FALSE);
 	if (flags & MAIN_MENU_USER)
-		create_user_menu(_("Favorites"), "apps",
+		create_user_menu(_("Favorites"), "applnk",
 				 root_menu, "gnome-favorites.png",
 				 fake_submenus, FALSE, FALSE, title, FALSE);
 	if (flags & MAIN_MENU_APPLETS)
 		create_applets_menu(root_menu, fake_submenus, title);
 
+#if 0
 	if (flags & MAIN_MENU_DISTRIBUTION &&
 	    distribution_info != NULL) {
 		if (distribution_info->menu_show_func)
@@ -5118,6 +5140,7 @@
 	if (flags & MAIN_MENU_KDE)
 		create_kde_menu(root_menu, fake_submenus, FALSE, FALSE,
 				title, FALSE);
+#endif
 
 	/*others here*/
 
@@ -5144,7 +5167,7 @@
 	}
 
 	if (flags & MAIN_MENU_USER_SUB) {
-		menu = create_user_menu(_("Favorites"), "apps", NULL,
+		menu = create_user_menu(_("Favorites"), "applnk", NULL,
 					"gnome-favorites.png",
 					fake_submenus, TRUE, TRUE, TRUE, FALSE);
 		menuitem = gtk_menu_item_new ();
@@ -5169,6 +5192,7 @@
 				   GTK_SIGNAL_FUNC(submenu_to_display),
 				   NULL);
 	}
+#if 0
 	if (flags & MAIN_MENU_DISTRIBUTION_SUB) {
 		add_distribution_submenu (root_menu, fake_submenus,
 					  FALSE /*launcher_add */);
@@ -5177,6 +5201,7 @@
 		add_kde_submenu (root_menu, fake_submenus,
 				 FALSE /*launcher_add */);
 	}
+#endif
 
 	menuitem = gtk_menu_item_new ();
 	gtk_widget_lock_accelerators (menuitem);
@@ -5297,7 +5322,7 @@
 	GtkWidget *wpanel = get_panel_parent(menu->button);
 	gboolean main_menu = (strcmp (menu->path, ".") == 0);
 	int flags;
-	const DistributionInfo *distribution_info = get_distribution_info ();
+/*	const DistributionInfo *distribution_info = get_distribution_info (); */
 
 	if (menu->global_main)
 		flags = global_config.menu_flags;
@@ -5315,11 +5340,13 @@
 
 		g_slist_free(list);
 	} else {
+#if 0
 		if(flags & MAIN_MENU_DISTRIBUTION &&
 		   ! (flags & MAIN_MENU_DISTRIBUTION_SUB) &&
 		   distribution_info != NULL &&
 		   distribution_info->menu_show_func)
 			distribution_info->menu_show_func(NULL, NULL);
+#endif
 
 		check_and_reread_applet(menu, main_menu);
 	}
diff -urN gnome-core-1.3.1.org/panel/menu.h gnome-core-1.3.1/panel/menu.h
--- gnome-core-1.3.1.org/panel/menu.h	Wed Feb 28 16:53:02 2001
+++ gnome-core-1.3.1/panel/menu.h	Wed Feb 28 16:53:33 2001
@@ -18,19 +18,13 @@
 	MAIN_MENU_USER_SUB = 1<<1,
 	MAIN_MENU_SYSTEM = 1<<2,
 	MAIN_MENU_SYSTEM_SUB = 1<<3,
-	MAIN_MENU_DISTRIBUTION = 1<<4,
-	MAIN_MENU_DISTRIBUTION_SUB = 1<<5,
-	MAIN_MENU_KDE = 1<<6,
-	MAIN_MENU_KDE_SUB = 1<<7,
 	/* keep this for compatibility. */
-	MAIN_MENU_OBSOLETE_DEBIAN = 1<<8,
-	MAIN_MENU_OBSOLETE_DEBIAN_SUB = 1<<9,
-	MAIN_MENU_APPLETS = 1<<10,
-	MAIN_MENU_APPLETS_SUB = 1<<11,
-	MAIN_MENU_PANEL = 1<<12,
-	MAIN_MENU_PANEL_SUB = 1<<13,
-	MAIN_MENU_DESKTOP = 1<<14,
-	MAIN_MENU_DESKTOP_SUB = 1<<15
+	MAIN_MENU_APPLETS = 1<<4,
+	MAIN_MENU_APPLETS_SUB = 1<<5,
+	MAIN_MENU_PANEL = 1<<7,
+	MAIN_MENU_PANEL_SUB = 1<<7,
+	MAIN_MENU_DESKTOP = 1<<8,
+	MAIN_MENU_DESKTOP_SUB = 1<<9  
 };
 
 /* this is defined in menu-properties.c */
diff -urN gnome-core-1.3.1.org/panel/panel-include.h gnome-core-1.3.1/panel/panel-include.h
--- gnome-core-1.3.1.org/panel/panel-include.h	Wed Feb 28 16:53:01 2001
+++ gnome-core-1.3.1/panel/panel-include.h	Wed Feb 28 16:53:33 2001
@@ -23,8 +23,6 @@
 #include "panel_config.h"
 #include "panel_config_global.h"
 
-#include "distribution.h"
-
 #include "menu.h"
 #include "menu-fentry.h"
 #include "menu-rh.h"
diff -urN gnome-core-1.3.1.org/panel/panel.c gnome-core-1.3.1/panel/panel.c
--- gnome-core-1.3.1.org/panel/panel.c	Wed Feb 28 16:53:02 2001
+++ gnome-core-1.3.1/panel/panel.c	Wed Feb 28 16:53:33 2001
@@ -52,9 +52,11 @@
 
 extern GtkTooltips *panel_tooltips;
 
+#if 0
 extern char *kde_menudir;
 extern char *kde_icondir;
 extern char *kde_mini_icondir;
+#endif
 
 
 /*the types of stuff we accept*/
@@ -950,6 +952,7 @@
 drop_menu (PanelWidget *panel, int pos, const char *dir)
 {
 	int flags = MAIN_MENU_SYSTEM | MAIN_MENU_USER;
+#if 0
 	DistributionType distribution = get_distribution_type ();
 
 	/*guess distribution menus*/
@@ -958,6 +961,7 @@
 	/* Guess KDE menus */
 	if(panel_file_exists(kde_menudir))
 		flags |= MAIN_MENU_KDE_SUB;
+#endif
 	load_menu_applet (dir, flags, TRUE, FALSE, NULL, panel, pos, TRUE);
 }
 
diff -urN gnome-core-1.3.1.org/panel/session.c gnome-core-1.3.1/panel/session.c
--- gnome-core-1.3.1.org/panel/session.c	Wed Feb 28 16:53:01 2001
+++ gnome-core-1.3.1/panel/session.c	Wed Feb 28 16:53:33 2001
@@ -44,9 +44,11 @@
 /*list of all panel widgets created*/
 extern GSList *panel_list;
 
+#if 0
 extern char *kde_menudir;
 extern char *kde_icondir;
 extern char *kde_mini_icondir;
+#endif
 
 int ss_cur_applet = 0;
 gboolean ss_done_save = FALSE;
@@ -865,10 +867,10 @@
 load_default_applets1(PanelWidget *panel)
 {
 	char *def_launchers[] =
-	      { "gnome/apps/gnome-help.desktop",
-		"gnome/apps/System/gnome-terminal.desktop",
-		"gnome/apps/Settings/gnomecc.desktop",
-		"gnome/apps/Internet/Netscape.desktop",
+	      { "applnk/gnome-help.desktop",
+		"applnk/System/gnome-terminal.desktop",
+		"applnk/Settings/GNOME/gnomecc.desktop",
+		"applnk/Internet/Netscape.desktop",
 		NULL };
 	int i;
 	char *p;
@@ -1040,7 +1042,7 @@
 {
 	GString *buf;
 	int count, num;	
-	DistributionType distribution = get_distribution_type();
+/*	DistributionType distribution = get_distribution_type(); */
 
 	count = gnome_config_get_int(PANEL_CONFIG_PATH
 				     "panel/Config/applet_count=0");
@@ -1197,12 +1199,14 @@
 					flags |= MAIN_MENU_SYSTEM_SUB|
 						MAIN_MENU_USER;
 				}
+#if 0
 				/*guess distribution menus*/
 				if(distribution != DISTRIBUTION_UNKNOWN)
 					flags |= MAIN_MENU_DISTRIBUTION_SUB;
 				/*guess KDE menus */
 				if(panel_file_exists(kde_menudir))
 					flags |= MAIN_MENU_KDE_SUB;
+#endif
 			}
 			if(old_style) {
 				/*this is needed to make panel properly
@@ -1213,6 +1217,7 @@
 				if(flags&MAIN_MENU_USER &&
 				   flags&MAIN_MENU_USER_SUB)
 					flags &=~ MAIN_MENU_USER;
+#if 0
 				if(flags&MAIN_MENU_DISTRIBUTION &&
 				   flags&MAIN_MENU_DISTRIBUTION_SUB)
 					flags &=~ MAIN_MENU_DISTRIBUTION;
@@ -1224,6 +1229,7 @@
 				if(flags&MAIN_MENU_KDE &&
 				   flags&MAIN_MENU_KDE_SUB)
 					flags &=~ MAIN_MENU_KDE;
+#endif
 				flags |= MAIN_MENU_APPLETS_SUB |
 					MAIN_MENU_PANEL_SUB |
 					MAIN_MENU_DESKTOP;
